Index: nvim/lazy/vim-apathy/after/ftplugin/javascript_apathy.vim
===================================================================
--- nvim.orig/lazy/vim-apathy/after/ftplugin/javascript_apathy.vim
+++ nvim/lazy/vim-apathy/after/ftplugin/javascript_apathy.vim
@@ -26,5 +26,8 @@ function! JavascriptNodeFind(target, cur
     catch
     endtry
   endif
+  if ! found
+      let target = luaeval("require'tsconfig'.includeexpr(_A)", a:target)
+  endif
   return target
 endfunction
