Index: .vim/bundle/asciidoc/ftplugin/asciidoc.vim
===================================================================
--- .vim.orig/bundle/asciidoc/ftplugin/asciidoc.vim
+++ .vim/bundle/asciidoc/ftplugin/asciidoc.vim
@@ -33,30 +33,30 @@ setlocal nosmartindent nocindent
 setlocal isk-=_
 
 " headings
-nnoremap <buffer> <leader>0 :call asciidoc#set_section_title_level(1)<cr>
-nnoremap <buffer> <leader>1 :call asciidoc#set_section_title_level(2)<cr>
-nnoremap <buffer> <leader>2 :call asciidoc#set_section_title_level(3)<cr>
-nnoremap <buffer> <leader>3 :call asciidoc#set_section_title_level(4)<cr>
-nnoremap <buffer> <leader>4 :call asciidoc#set_section_title_level(5)<cr>
+" nnoremap <buffer> <leader>0 :call asciidoc#set_section_title_level(1)<cr>
+" nnoremap <buffer> <leader>1 :call asciidoc#set_section_title_level(2)<cr>
+" nnoremap <buffer> <leader>2 :call asciidoc#set_section_title_level(3)<cr>
+" nnoremap <buffer> <leader>3 :call asciidoc#set_section_title_level(4)<cr>
+" nnoremap <buffer> <leader>4 :call asciidoc#set_section_title_level(5)<cr>
 
 " TODO: Make simple 'j/k' offsets honour setext style sections
-nnoremap <buffer> <expr><silent> [[ asciidoc#find_prior_section_title()
-nnoremap <buffer> <expr><silent> [] asciidoc#find_prior_section_title() . 'j'
-nnoremap <buffer> <expr><silent> ]] asciidoc#find_next_section_title()
-nnoremap <buffer> <expr><silent> ][ asciidoc#find_next_section_title() . 'k'
-
-xnoremap <buffer> <expr><silent> [[ asciidoc#find_prior_section_title()
-xnoremap <buffer> <expr><silent> [] asciidoc#find_prior_section_title() . 'j'
-xnoremap <buffer> <expr><silent> ]] asciidoc#find_next_section_title()
-xnoremap <buffer> <expr><silent> ][ asciidoc#find_next_section_title() . 'k'
-
-xnoremap <buffer> <silent> lu :call asciidoc#make_list('*')<cr>gv
-xnoremap <buffer> <silent> lo :call asciidoc#make_list('.')<cr>gv
-xnoremap <buffer> <silent> l< :call asciidoc#dent_list('in')<cr>gv
-xnoremap <buffer> <silent> l> :call asciidoc#dent_list('out')<cr>gv
+" nnoremap <buffer> <expr><silent> [[ asciidoc#find_prior_section_title()
+" nnoremap <buffer> <expr><silent> [] asciidoc#find_prior_section_title() . 'j'
+" nnoremap <buffer> <expr><silent> ]] asciidoc#find_next_section_title()
+" nnoremap <buffer> <expr><silent> ][ asciidoc#find_next_section_title() . 'k'
+
+" xnoremap <buffer> <expr><silent> [[ asciidoc#find_prior_section_title()
+" xnoremap <buffer> <expr><silent> [] asciidoc#find_prior_section_title() . 'j'
+" xnoremap <buffer> <expr><silent> ]] asciidoc#find_next_section_title()
+" xnoremap <buffer> <expr><silent> ][ asciidoc#find_next_section_title() . 'k'
+
+" xnoremap <buffer> <silent> lu :call asciidoc#make_list('*')<cr>gv
+" xnoremap <buffer> <silent> lo :call asciidoc#make_list('.')<cr>gv
+" xnoremap <buffer> <silent> l< :call asciidoc#dent_list('in')<cr>gv
+" xnoremap <buffer> <silent> l> :call asciidoc#dent_list('out')<cr>gv
 
-nmap     <buffer> <leader>lu viplu<c-\><c-n>``
-nmap     <buffer> <leader>lo viplo<c-\><c-n>``
+" nmap     <buffer> <leader>lu viplu<c-\><c-n>``
+" nmap     <buffer> <leader>lo viplo<c-\><c-n>``
 
 let s:asciidoc = {}
 let s:asciidoc.delimited_block_pattern = '^[-.~_+^=*\/]\{4,}\s*$'
@@ -155,8 +155,8 @@ endfunction
 " Easily reflow text
 " the  Q form (badly) tries to keep cursor position
 " the gQ form subsequently jumps over the reformatted block
-nnoremap <silent> <buffer> Q  :call <SID>Q(0)<cr>
-nnoremap <silent> <buffer> gQ :call <SID>Q(1)<cr>
+" nnoremap <silent> <buffer> Q  :call <SID>Q(0)<cr>
+" nnoremap <silent> <buffer> gQ :call <SID>Q(1)<cr>
 
 function! s:Q(skip_block_after_format)
   if ! a:skip_block_after_format
