Index: .vim/pack/ftplugins/start/asciidoc/ftplugin/asciidoc.vim
===================================================================
--- .vim.orig/pack/ftplugins/start/asciidoc/ftplugin/asciidoc.vim
+++ .vim/pack/ftplugins/start/asciidoc/ftplugin/asciidoc.vim
@@ -120,7 +120,7 @@ endif
 
 " indent
 " ------
-setlocal indentexpr=GetAsciidocIndent()
+" setlocal indentexpr=GetAsciidocIndent()
 
 " stolen from the RST equivalent
 function! GetAsciidocIndent()
@@ -188,7 +188,7 @@ function! s:Q(skip_block_after_format)
   endif
 endfunction
 
-setlocal formatexpr=AsciidocFormatexpr()
+" setlocal formatexpr=AsciidocFormatexpr()
 
 function! AsciidocFormatexpr()
   return s:asciidoc.formatexpr()
