Index: .vim/ipi/buffergator/autoload/buffergator.vim
===================================================================
--- .vim.orig/ipi/buffergator/autoload/buffergator.vim
+++ .vim/ipi/buffergator/autoload/buffergator.vim
@@ -580,8 +580,10 @@ function! s:NewCatalogViewer(name, title
     " Opens viewer if closed, closes viewer if open.
     function! l:catalog_viewer.toggle() dict
         " get buffer number of the catalog view buffer, creating it if neccessary
+        let l:open = 0
         if self.bufnum < 0 || !bufexists(self.bufnum)
             call self.open()
+            let l:open = 1
         else
             let l:bfwn = bufwinnr(self.bufnum)
             if l:bfwn >= 0
@@ -590,6 +592,9 @@ function! s:NewCatalogViewer(name, title
                 call self.open()
             endif
         endif
+        if l:open
+            nnoremap <buffer> <silent> <Esc> :BuffergatorClose<CR>
+        endif
     endfunction
 
     " Creates a new buffer, renders and opens it.
