Index: .vim/bundle/buffergator/plugin/buffergator.vim
===================================================================
--- .vim.orig/bundle/buffergator/plugin/buffergator.vim	2011-10-07 12:40:00.000000000 +0200
+++ .vim/bundle/buffergator/plugin/buffergator.vim	2011-10-07 12:42:03.000000000 +0200
@@ -541,6 +541,7 @@
             elseif g:buffergator_viewport_split_policy =~ '[TtBb]' && g:buffergator_split_size
                 execute("resize " . g:buffergator_split_size)
             endif
+            setlocal winfixheight
         endif
     endfunction
 
@@ -1143,7 +1144,7 @@
             " explicit split command not given: switch to buffer in current
             " window
             let &switchbuf="useopen"
-            execute("silent keepalt keepjumps buffer " . a:bufnum)
+            execute("silent buffer " . a:bufnum)
         else
             " explcit split command given: split current window
             let &switchbuf="split"
@@ -1521,12 +1522,12 @@
     let l:line = line(".")
     let l:status_line = "[[buffergator]]"
     if has_key(b:buffergator_catalog_viewer.jump_map, l:line)
-        let l:status_line .= " Buffer " . string(l:line) . " of " . string(len(b:buffergator_catalog_viewer.buffers_catalog))
+        let l:status_line .= " Buffer " . string(l:line) . " of " . string(len(b:buffergator_catalog_viewer.buffers_catalog)) . " | v(s)plit spl(i)t (t)ab (d)elete (x)wipe (A)zoom (r)efresh (cd)display (cs)sort"
     endif
     return l:status_line
 endfunction
 function! BuffergatorTabsStatusLine()
-    let l:status_line = "[[buffergator: tabs]]"
+    let l:status_line = "[[buffergator: tabs]] <C-n>ext <C-p>revious (r)efresh (A)zoom (cd)display"
     return l:status_line
 endfunction
 " 1}}}
