Index: vim/.vim/bundle/cdargs/plugin/cdargs.vim
===================================================================
--- vim.orig/.vim/bundle/cdargs/plugin/cdargs.vim	2010-07-23 22:27:30.086445714 +0200
+++ vim/.vim/bundle/cdargs/plugin/cdargs.vim	2010-07-24 09:32:41.912704287 +0200
@@ -203,6 +203,15 @@
   end
 endfunction
 
+function! s:add(description)
+  " Add current directory to bookmarks
+  if strlen(a:raw) && strlen(getcwd())
+    echo system("cdargs -a ':".a:raw.":".getcwd()."'")
+  else
+    echom "Failed to add bookmark, description or current working directory not available."
+  end
+endfunction
+
 function! s:execute(command, raw)
   " Escape spaces in the path
   let l:path = substitute(s:path_for(a:raw), " ", '\\ ', "g")
@@ -210,6 +219,9 @@
   if strlen(l:path)
     try
       execute a:command . ' ' . l:path
+      if index(['cd', 'lcd'], a:command) != -1
+          echo "Current working directory: ".getcwd()
+      endif
     catch
       call s:error('Unable to find path: ' . l:path)
     endtry
@@ -218,9 +230,13 @@
 
 " Change working directory to bookmark or it's subpath
 command! -nargs=1 -complete=customlist,s:directory_completion Cdb call s:execute('cd', <f-args>)
+command! -nargs=1 -complete=customlist,s:directory_completion Lcdb call s:execute('lcd', <f-args>)
 
 " Edit file under a bookmark's path
 command! -nargs=1 -complete=customlist,s:file_completion Eb call s:execute('edit', <f-args>)
 
 " Edit file under a bookmark's path in a new tab
 command! -nargs=1 -complete=customlist,s:file_completion Tb call s:execute('tabedit', <f-args>)
+
+" Add current directory to bookmarks
+command! -nargs=1 Ca call s:add(<q-args>)
