Index: vim/.vim/plugin/cdargs.vim
===================================================================
--- vim.orig/.vim/plugin/cdargs.vim	2010-06-14 22:31:39.000000000 +0200
+++ vim/.vim/plugin/cdargs.vim	2010-06-14 22:35:45.000000000 +0200
@@ -203,6 +203,15 @@
   end
 endfunction
 
+function! s:add(description)
+  " Add current directory to bookmarks
+  if strlen(a:raw) && strlen(getcwd())
+    echo system("cdargs -a ':".a:raw.":".getcwd()."'")
+  else
+    echom "Failed to add bookmark, description or current working directory not available."
+  end
+endfunction
+
 function! s:execute(command, raw)
   " Escape spaces in the path
   let l:path = substitute(s:path_for(a:raw), " ", '\\ ', "g")
@@ -224,3 +233,6 @@
 
 " Edit file under a bookmark's path in a new tab
 command! -nargs=1 -complete=customlist,s:file_completion Tb call s:execute('tabedit', <f-args>)
+
+" Add current directory to bookmarks
+command! -nargs=1 Ca call s:add(<q-args>)
