Index: .vim/bundle/chapa/ftplugin/python/chapa.vim
===================================================================
--- .vim.orig/bundle/chapa/ftplugin/python/chapa.vim	2011-05-14 08:45:32.396044576 +0200
+++ .vim/bundle/chapa/ftplugin/python/chapa.vim	2011-05-14 12:18:28.993186547 +0200
@@ -13,10 +13,6 @@
 endif
 
 "{{{ Default Mappings 
-" Set local folding settings
-setlocal foldmethod=manual
-setlocal foldtext=ChapaCustomFoldText()
-
 
 if (exists('g:chapa_default_mappings'))
     if (! exists('g:chapa_no_repeat_mappings'))
@@ -69,21 +65,6 @@
     nmap cif <Plug>ChapaCommentThisFunction
     nmap cnf <Plug>ChapaCommentNextFunction
     nmap cpf <Plug>ChapaCommentPreviousFunction
-
-    " Folding Method
-    nmap zim <Plug>ChapaFoldThisMethod
-    nmap znm <Plug>ChapaFoldNextMethod
-    nmap zpm <Plug>ChapaFoldPreviousMethod
-
-    " Folding Class
-    nmap zic <Plug>ChapaFoldThisClass
-    nmap znc <Plug>ChapaFoldNextClass
-    nmap zpc <Plug>ChapaFoldPreviousClass
-
-    " Folding Function
-    nmap zif <Plug>ChapaFoldThisFunction
-    nmap znf <Plug>ChapaFoldNextFunction
-    nmap zpf <Plug>ChapaFoldPreviousFunction
 endif
 
 "{{{ Helpers
