Index: vim/.vim/plugin/command-t.vim
===================================================================
--- vim.orig/.vim/plugin/command-t.vim	2010-03-24 19:52:02.000000000 +0100
+++ vim/.vim/plugin/command-t.vim	2010-03-24 19:52:07.000000000 +0100
@@ -30,7 +30,7 @@
 command CommandT :call <SID>CommandTShow()
 command CommandTFlush :call <SID>CommandTFlush()
 
-silent! nmap <unique> <silent> <Leader>t :CommandT<CR>
+"silent! nmap <unique> <silent> <Leader>t :CommandT<CR>
 
 function s:CommandTRubyWarning()
   echohl WarningMsg
