Index: vim/bundle/ctrlp/autoload/ctrlp.vim
===================================================================
--- vim.orig/bundle/ctrlp/autoload/ctrlp.vim	2014-01-18 12:20:30.761736715 +0100
+++ vim/bundle/ctrlp/autoload/ctrlp.vim	2014-01-18 13:18:50.377833085 +0100
@@ -713,6 +713,17 @@
 			let str = fnamemodify(s:fnesc(spc, 'g'), mdr).nmd
 		en
 	en
+	let str_tmp = str
+	if str == ''
+		 let dir = s:headfirstntail(s:fnesc(ctrlp#getcline(), 'g'))
+		 if len(dir) == 2 && dir[0] != '' && isdirectory(dir[0])
+				cal ctrlp#setdir(dir[0])
+				cal ctrlp#switchtype(0)
+				cal ctrlp#recordhist()
+				cal s:PrtClear()
+				return
+		 en
+	en
 	if str == '' | retu | en
 	unl! s:hstgot
 	let s:act_add = 1
@@ -726,6 +737,29 @@
 	elsei len(dirs) > 1
 		let str .= s:findcommon(dirs, str)
 	en
+	if str_tmp == str
+		 let slash = s:lash()
+		 let dir = s:headfirstntail(s:fnesc(ctrlp#getcline(), 'g'))
+		 if len(dir) == 2 && dir[0] != '' && isdirectory(dir[0])
+			  let exp = 1
+			  while exp
+				   if str[:len(dir[0])] != dir[0]
+					    let exp = 0
+				   en
+				   let tmp = s:headfirstntail(dir[1])
+				   if len(tmp) == 2 && dir[0] != ''
+					    let dir = [dir[0].slash.tmp[0], tmp[1]]
+				   el
+					    break
+				   en
+			  endw
+				cal ctrlp#setdir(dir[0])
+				cal ctrlp#switchtype(0)
+				cal ctrlp#recordhist()
+				cal s:PrtClear()
+				return
+		 en
+	en
 	let s:prompt[0] = exists('hasat') ? hasat[0].str : str
 	cal s:BuildPrompt(1)
 	unl s:act_add
@@ -1511,6 +1545,13 @@
 	retu len(parts) == 1 ? ['', parts[0]] : len(parts) == 2 ? parts : []
 endf
 
+fu! s:headfirstntail(str)
+	" seems to be a bug in vim
+	" let parts = split(a:str, '^[^\/]\+\zs[\/]')
+	let parts = matchlist(a:str, '^\([^\/]\+\)\zs[\/]\(.*\)$')[1:2]
+	retu len(parts) == 1 ? ['', parts[0]] : len(parts) == 2 ? parts : []
+endf
+
 fu! s:lash(...)
 	retu ( a:0 ? a:1 : s:dyncwd ) !~ '[\/]$' ? s:lash : ''
 endf
