Index: .vim/pack/submodules/start/cursorword/autoload/cursorword.vim
===================================================================
--- .vim.orig/pack/submodules/start/cursorword/autoload/cursorword.vim
+++ .vim/pack/submodules/start/cursorword/autoload/cursorword.vim
@@ -29,7 +29,7 @@ function! cursorword#matchadd() abort
   silent! call matchdelete(w:cursorword_id0)
   silent! call matchdelete(w:cursorword_id1)
   let w:cursorword_match = 0
-  if !enable || word ==# '' || len(word) !=# strchars(word) | return | endif
+  if !enable || word ==# '' | return | endif
   let pattern = '\<' . escape(word, '~"\.^$[]*') . '\>'
   let w:cursorword_id0 = matchadd('CursorWord0', pattern, -1)
   let w:cursorword_id1 = matchadd('CursorWord' . &l:cursorline, '\%' . linenr . 'l' . pattern, -1)
