Index: vim/.vim/ftplugin/debchangelog.vim
===================================================================
--- vim.orig/.vim/ftplugin/debchangelog.vim	2010-07-10 10:30:53.000000000 +0200
+++ vim/.vim/ftplugin/debchangelog.vim	2010-07-10 10:31:01.000000000 +0200
@@ -228,7 +228,7 @@
 
 augroup changelogMenu
 au BufEnter * if &filetype == "debchangelog" | call <SID>MakeMenu() | endif
-au BufLeave * if &filetype == "debchangelog" | aunmenu Changelog | endif
+au BufLeave * if &filetype == "debchangelog" | silent! aunmenu Changelog | endif
 augroup END
 
 " }}}
