Index: vim/.vim/autoload/delimitMate.vim
===================================================================
--- vim.orig/.vim/autoload/delimitMate.vim	2010-05-13 18:11:55.000000000 +0200
+++ vim/.vim/autoload/delimitMate.vim	2010-05-13 18:19:17.000000000 +0200
@@ -224,6 +224,11 @@
 function! delimitMate#JumpIn(char) " {{{
 	let line = getline('.')
 	let col = col('.')-2
+	if line[col + 1] =~ '[a-zA-Z0-9]' && b:delimitMate_smart_matchpairs
+		" cursor is at the beginning or in the middle of a string
+		" so don't insert a pair
+		return ''
+	endif
 	if (col) < 0
 		call setline('.',a:char.line)
 		call insert(b:delimitMate_buffer, a:char)
@@ -296,6 +301,10 @@
 		" Get out of the string.
 		"return delimitMate#WriteBefore(a:char)
 		return a:char . delimitMate#Del()
+	elseif line[col] =~ '[a-zA-Z0-9]' || line[col + 1] =~ '[a-zA-Z0-9]' && b:delimitMate_smart_quotes
+		" cursor is in the at the beginning or in the middle of a string
+		" so don't insert a pair
+		return a:char
 	elseif (line[col] =~ '[a-zA-Z0-9]' && a:char == "'") ||
 				\(line[col] =~ '[a-zA-Z0-9]' && b:delimitMate_smart_quotes)
 		" Seems like an apostrophe or a closing, insert a single quote.
