Index: vim/.vim/autoload/delimitMate.vim
===================================================================
--- vim.orig/.vim/autoload/delimitMate.vim	2010-06-07 15:26:24.666091766 +0200
+++ vim/.vim/autoload/delimitMate.vim	2010-06-07 15:27:17.599390276 +0200
@@ -357,6 +357,11 @@
 	endif
 	let line = getline('.')
 	let col = col('.')-2
+	if line[col + 1] =~ '[a-zA-Z0-9]'
+		" cursor is at the beginning or in the middle of a string
+		" so don't insert a pair
+		return ''
+	endif
 	if (col) < 0
 		call setline('.',a:char.line)
 		call insert(b:delimitMate_buffer, a:char)
@@ -449,6 +454,10 @@
 		" Get out of the string.
 		"return delimitMate#WriteBefore(a:char)
 		return a:char . delimitMate#Del()
+	elseif line[col] =~ '[a-zA-Z0-9]' || line[col + 1] =~ '[a-zA-Z0-9]' && b:delimitMate_smart_quotes
+		" cursor is in the at the beginning or in the middle of a string
+		" so don't insert a pair
+		return a:char
 	elseif (line[col] =~ '[a-zA-Z0-9]' && a:char == "'") ||
 				\(line[col] =~ '[a-zA-Z0-9]' && b:delimitMate_smart_quotes)
 		" Seems like an apostrophe or a closing, insert a single quote.
