Index: vimconfig/.vim/autoload/delimitMate.vim
===================================================================
--- vimconfig.orig/.vim/autoload/delimitMate.vim	2010-08-02 09:07:38.000000000 +0200
+++ vimconfig/.vim/autoload/delimitMate.vim	2010-08-03 10:04:54.000000000 +0200
@@ -434,6 +434,11 @@
 	endif
 	let line = getline('.')
 	let col = col('.')-2
+	if line[col + 1] =~ '[[:alnum:]]'
+		" cursor is at the beginning or in the middle of a string
+		" so don't insert a pair
+		return ''
+	endif
 	if (col) < 0
 		call setline('.',a:char.line)
 		call insert(b:_l_delimitMate_buffer, a:char)
@@ -458,6 +463,10 @@
 				\ index(b:_l_delimitMate_nesting_quotes, a:char) < 0
 		" Get out of the string.
 		return a:char . delimitMate#Del()
+	elseif line[col] =~ '[[:alnum:]]' || line[col + 1] =~ '[[:alnum:]]' && exists("b:delimitMate_smart_quotes") && b:delimitMate_expand_space
+		" cursor is in the at the beginning or in the middle of a string
+		" so don't insert a pair
+		return a:char
 	elseif (line[col] =~ '[[:alnum:]]' && a:char == "'") ||
 				\ (b:_l_delimitMate_smart_quotes &&
 				\ (line[col] =~ '[[:alnum:]]' ||
