Index: vimconfig/.vim/autoload/delimitMate.vim
===================================================================
--- vimconfig.orig/.vim/autoload/delimitMate.vim	2010-05-18 08:30:06.000000000 +0200
+++ vimconfig/.vim/autoload/delimitMate.vim	2010-05-20 08:43:23.000000000 +0200
@@ -342,6 +342,11 @@
 function! delimitMate#JumpIn(char) " {{{
 	let line = getline('.')
 	let col = col('.')-2
+	if line[col + 1] =~ '[a-zA-Z0-9]'
+		" cursor is at the beginning or in the middle of a string
+		" so don't insert a pair
+		return ''
+	endif
 	if (col) < 0
 		call setline('.',a:char.line)
 		call insert(b:delimitMate_buffer, a:char)
@@ -419,6 +424,10 @@
 		" Get out of the string.
 		"return delimitMate#WriteBefore(a:char)
 		return a:char . delimitMate#Del()
+	elseif line[col] =~ '[a-zA-Z0-9]' || line[col + 1] =~ '[a-zA-Z0-9]' && b:delimitMate_smart_quotes
+		" cursor is in the at the beginning or in the middle of a string
+		" so don't insert a pair
+		return a:char
 	elseif (line[col] =~ '[a-zA-Z0-9]' && a:char == "'") ||
 				\(line[col] =~ '[a-zA-Z0-9]' && b:delimitMate_smart_quotes)
 		" Seems like an apostrophe or a closing, insert a single quote.
