Index: .vim/bundle/asciidoc/ftdetect/asciidoc_filetype.vim
===================================================================
--- .vim.orig/bundle/asciidoc/ftdetect/asciidoc_filetype.vim	2010-12-19 23:28:02.344852911 +0100
+++ .vim/bundle/asciidoc/ftdetect/asciidoc_filetype.vim	2010-12-19 23:30:10.412786911 +0100
@@ -10,8 +10,8 @@
 " The first sets asciidoc syntax highlighting on all .txt files, the second
 " only existing files *.txt that appear to be AsciiDoc files.
 
-au BufNewFile,BufRead *.txt  setfiletype asciidoc
-"au BufRead *.txt call s:FTasciidoc()
+"au BufNewFile,BufRead *.txt  setfiletype asciidoc
+au BufRead *.txt call s:FTasciidoc()
 
 " This function checks for a valid AsciiDoc document title after first
 " skipping any leading comments.
