Index: vim/.vim/addons/FuzzyFinder/plugin/fuf.vim
===================================================================
--- vim.orig/.vim/addons/FuzzyFinder/plugin/fuf.vim	2009-10-31 16:08:06.000000000 +0100
+++ vim/.vim/addons/FuzzyFinder/plugin/fuf.vim	2009-10-31 16:08:15.000000000 +0100
@@ -8,7 +8,7 @@
 if exists('g:loaded_fuf')
   finish
 elseif v:version < 702
-  echoerr 'FuzzyFinder does not support this version of vim (' . v:version . ').'
+  "echoerr 'FuzzyFinder does not support this version of vim (' . v:version . ').'
   finish
 endif
 let g:loaded_fuf = 1
