Index: .vim/pack/vimscripts/opt/FuzzyFinder/plugin/fuf.vim
===================================================================
--- .vim.orig/pack/vimscripts/opt/FuzzyFinder/plugin/fuf.vim
+++ .vim/pack/vimscripts/opt/FuzzyFinder/plugin/fuf.vim
@@ -48,6 +48,7 @@ function s:initialize()
   call l9#defineVariableDefault('g:fuf_previewHeight'    , 0)
   call l9#defineVariableDefault('g:fuf_autoPreview'      , 0)
   call l9#defineVariableDefault('g:fuf_useMigemo'        , 0)
+  call l9#defineVariableDefault('g:fuf_scratch_location' , 'leftabove')
   "---------------------------------------------------------------------------
   call l9#defineVariableDefault('g:fuf_buffer_prompt'     , '>Buffer[]>')
   call l9#defineVariableDefault('g:fuf_buffer_switchOrder', 10)
Index: .vim/pack/vimscripts/opt/FuzzyFinder/autoload/fuf.vim
===================================================================
--- .vim.orig/pack/vimscripts/opt/FuzzyFinder/autoload/fuf.vim
+++ .vim/pack/vimscripts/opt/FuzzyFinder/autoload/fuf.vim
@@ -710,7 +710,7 @@ function s:activateFufBuffer()
   "         if 'autochdir' was set on.
   lcd .
   let cwd = getcwd()
-  call l9#tempbuffer#openScratch(s:FUF_BUF_NAME, 'fuf', [], 1, 0, 1, {})
+  call l9#tempbuffer#openScratch(s:FUF_BUF_NAME, 'fuf', [], g:fuf_scratch_location, 0, 1, {})
   resize 1 " for issue #21 
   " lcd ... : countermeasure against auto-cd script
   lcd `=cwd`
