Index: vim/.vim/plugin/fuf.vim
===================================================================
--- vim.orig/.vim/plugin/fuf.vim	2009-10-17 10:37:26.000000000 +0200
+++ vim/.vim/plugin/fuf.vim	2009-10-17 10:38:17.000000000 +0200
@@ -48,6 +48,7 @@
   call s:defineOption('g:fuf_enumeratingLimit', 50)
   call s:defineOption('g:fuf_maxMenuWidth'    , 78)
   call s:defineOption('g:fuf_useMigemo'       , 0)
+  call s:defineOption('g:fuf_onelinebuf_location', 'leftabove')
   "---------------------------------------------------------------------------
   call s:defineOption('g:fuf_buffer_prompt'     , '>Buffer>')
   call s:defineOption('g:fuf_buffer_switchOrder', 10)
Index: vim/.vim/autoload/fuf.vim
===================================================================
--- vim.orig/.vim/autoload/fuf.vim	2009-10-17 10:37:26.000000000 +0200
+++ vim/.vim/autoload/fuf.vim	2009-10-17 10:38:17.000000000 +0200
@@ -623,12 +623,22 @@
 
 "
 function s:openFufBuffer()
+  let openbuffer = 'topleft'
+  "if g:fuf_onelinebuf_location =~ '^top\(l\(e\(f\(t\)\?\)\?\)\?\)\?$'
+	"let openbuffer = 'topleft'
+  if g:fuf_onelinebuf_location =~ '^lefta\(b\(a\(v\(e\)\?\)\?\)\?\)\?$' || g:fuf_onelinebuf_location =~ '^abo\(o\(v\(e\(l\(e\(f\(t\)\?\)\?\)\?\)\?\)\?\)\?\)\?$'
+    let openbuffer = 'leftabove'
+  elseif g:fuf_onelinebuf_location =~ '^rightb\(e\(l\(o\(w\)\?\)\?\)\?\)\?$' || g:fuf_onelinebuf_location =~ '^bel\(o\(w\(r\(i\(g\(h\(t\)\?\)\?\)\?\)\?\)\?\)\?\)\?$'
+    let openbuffer = 'rightbelow'
+  elseif g:fuf_onelinebuf_location =~ '^bot\(r\(i\(g\(h\(t\)\?\)\?\)\?\)\?\)\?$'
+    let openbuffer = 'botright'
+  endif
   if !bufexists(s:bufNrFuf)
-    topleft 1new
+    execute openbuffer.' 1new'
     file `='[fuf]'`
     let s:bufNrFuf = bufnr('%')
   elseif bufwinnr(s:bufNrFuf) == -1
-    topleft 1split
+    execute openbuffer.' 1split'
     execute s:bufNrFuf . 'buffer'
     delete _
   elseif bufwinnr(s:bufNrFuf) != bufwinnr('%')
