Index: vim/.vim/plugin/fuf.vim
===================================================================
--- vim.orig/.vim/plugin/fuf.vim	2009-07-04 12:38:05.000000000 +0200
+++ vim/.vim/plugin/fuf.vim	2009-07-04 13:31:48.000000000 +0200
@@ -1446,17 +1446,25 @@
 function! s:WindowManager.deactivate()
   if exists(':AutoComplPopUnlock') | execute ':AutoComplPopUnlock' | endif
   " must close after returning to previous window
-  wincmd j
+  wincmd p
   execute self.buf_nr . 'bdelete'
 endfunction
 
 " Returns a buffer number. Creates new buffer if a:buf_nr is a invalid number
 function! s:Open1LineBuffer(buf_nr, buf_name)
+  let openbuffer = 'leftabove'
+  if g:FuzzyFinderOptions.Base.onelinebuf_location =~ '^top\(l\(e\(f\(t\)\?\)\?\)\?\)\?$'
+    let openbuffer = 'topleft'
+  elseif g:FuzzyFinderOptions.Base.onelinebuf_location =~ '^right\(b\(e\(l\(o\(w\)\?\)\?\)\?\)\?\)\?$' || g:FuzzyFinderOptions.Base.onelinebuf_location =~ '^below\(r\(i\(g\(h\(t\)\?\)\?\)\?\)\?\)\?$'
+    let openbuffer = 'rightbelow'
+  elseif g:FuzzyFinderOptions.Base.onelinebuf_location =~ '^bot\(r\(i\(g\(h\(t\)\?\)\?\)\?\)\?\)\?$'
+    let openbuffer = 'botright'
+  endif
   if !bufexists(a:buf_nr)
-    leftabove 1new
+    execute openbuffer.' 1new'
     execute printf('file `=%s`', string(a:buf_name))
   elseif bufwinnr(a:buf_nr) == -1
-    leftabove 1split
+    execute openbuffer.' 1split'
     execute a:buf_nr . 'buffer'
     delete _
   elseif bufwinnr(a:buf_nr) != bufwinnr('%')
@@ -1596,6 +1604,7 @@
 let g:FuzzyFinderOptions.Base.max_menu_width     = 80
 let g:FuzzyFinderOptions.Base.lasting_cache      = 1
 let g:FuzzyFinderOptions.Base.migemo_support     = 0
+let g:FuzzyFinderOptions.Base.onelinebuf_location  = 'leftabove'
 "-----------------------------------------------------------------------------
 let g:FuzzyFinderOptions.Buffer.mode_available   = 1
 let g:FuzzyFinderOptions.Buffer.prompt           = '>Buffer>'
