Index: .vim/bundle/gnupg/plugin/gnupg.vim
===================================================================
--- .vim.orig/bundle/gnupg/plugin/gnupg.vim	2011-07-12 14:21:26.000000000 +0200
+++ .vim/bundle/gnupg/plugin/gnupg.vim	2011-07-12 14:21:45.000000000 +0200
@@ -134,7 +134,7 @@
 
 " check for correct vim version {{{2
 if (v:version < 702)
-  echohl ErrorMsg | echo 'plugin gnupg.vim requires Vim version >= 7.2' | echohl None
+  "echohl ErrorMsg | echo 'plugin gnupg.vim requires Vim version >= 7.2' | echohl None
   finish
 endif
 
