Index: .vim/bundle/gundo/plugin/gundo.vim
===================================================================
--- .vim.orig/bundle/gundo/plugin/gundo.vim	2010-12-19 23:27:26.390838910 +0100
+++ .vim/bundle/gundo/plugin/gundo.vim	2010-12-19 23:30:39.778096910 +0100
@@ -17,7 +17,7 @@
 let loaded_gundo = 1"}}}
 
 if v:version < '703'"{{{
-    echo  "Gundo requires Vim 7.3+"
+    "echo  "Gundo requires Vim 7.3+"
     finish
 endif"}}}
 
