Index: .vim/bundle/gundo/plugin/gundo.vim
===================================================================
--- .vim.orig/bundle/gundo/plugin/gundo.vim	2011-03-19 22:07:35.000000000 +0100
+++ .vim/bundle/gundo/plugin/gundo.vim	2011-03-19 22:14:35.122786407 +0100
@@ -18,7 +18,7 @@
 
 if v:version < '703'"{{{
     function! s:GundoDidNotLoad()
-        echohl WarningMsg|echomsg "Gundo unavailable: requires Vim 7.3+"|echohl None
+        "echohl WarningMsg|echomsg "Gundo unavailable: requires Vim 7.3+"|echohl None
     endfunction
     command! -nargs=0 GundoToggle call s:GundoDidNotLoad()
     finish
