Index: .vim/bundle/gundo/plugin/gundo.vim
===================================================================
--- .vim.orig/bundle/gundo/plugin/gundo.vim	2010-12-01 17:50:25.000000000 +0100
+++ .vim/bundle/gundo/plugin/gundo.vim	2010-12-01 17:50:40.000000000 +0100
@@ -17,6 +17,7 @@
 let loaded_gundo = 1"}}}
 
 if v:version < '703'"{{{
+    finish
     echo  "Gundo requires Vim 7.3+"
     finish
 endif"}}}
