Index: .vim/bundle/highlight/plugin/highlight.vim
===================================================================
--- .vim.orig/bundle/highlight/plugin/highlight.vim	2010-12-19 23:27:26.834616909 +0100
+++ .vim/bundle/highlight/plugin/highlight.vim	2010-12-19 23:30:37.019476910 +0100
@@ -112,6 +112,14 @@
    let g:lcolor_fg = "white,white,black,black,black,black,white,white"
 endif
 
+if !exists('g:cterm_lcolor_bg')
+   let g:cterm_lcolor_bg = "yellow,green,magenta,lightblue,lightred,lightgreen,darkgray,cyan"
+endif
+
+if !exists('g:cterm_lcolor_fg')
+   let g:cterm_lcolor_fg = "black,black,white,black,black,black,white,black"
+endif
+
 " Define colors for Pattern highlight
 if !exists('g:pcolor_bg')
    let g:pcolor_bg = "yellow,blue,green,magenta,cyan,brown,orange,red"
@@ -121,6 +129,14 @@
    let g:pcolor_fg = "black,white,black,white,black,white,black,white"
 endif
 
+if !exists('g:cterm_pcolor_bg')
+   let g:cterm_pcolor_bg = "yellow,blue,green,magenta,lightcyan,brown,lightyellow,red"
+endif
+
+if !exists('g:cterm_pcolor_fg')
+   let g:cterm_pcolor_fg = "black,white,black,white,black,white,black,white"
+endif
+
 " Highlight: Highlight line or pattern 
 function! <SID>Highlight(mode)
    " Line mode
@@ -222,13 +238,17 @@
    let s:lcolor_n = 0
 
    let s:lcolor_max = s:Min(s:ItemCount(g:lcolor_bg . ','), s:ItemCount(g:lcolor_fg . ','))
+   let s:cterm_lcolor_max = s:Min(s:ItemCount(g:cterm_lcolor_bg . ','), s:ItemCount(g:cterm_lcolor_fg . ','))
 
    let ci = 0
-   while ci < s:lcolor_max
+   while ci < s:lcolor_max && ci < s:cterm_lcolor_max
       let bgColor = s:Strntok(g:lcolor_bg, ',', ci + 1)
       let fgColor = s:Strntok(g:lcolor_fg, ',', ci + 1)
+      let cterm_bgColor = s:Strntok(g:cterm_lcolor_bg, ',', ci + 1)
+      let cterm_fgColor = s:Strntok(g:cterm_lcolor_fg, ',', ci + 1)
      
       exec 'hi ' . s:lcolor_grp . ci .
+         \ ' ctermfg=' . cterm_fgColor . ' ctermbg=' . cterm_bgColor .
          \ ' guifg =' . fgColor . ' guibg=' . bgColor
      
       let ci = ci + 1
@@ -243,14 +263,18 @@
    let s:pcolor_n = 0
 
    let s:pcolor_max = s:Min(s:ItemCount(g:pcolor_bg . ','), s:ItemCount(g:pcolor_fg . ','))
+   let s:cterm_lcolor_max = s:Min(s:ItemCount(g:cterm_pcolor_bg . ','), s:ItemCount(g:cterm_pcolor_fg . ','))
 
    let ci = 0
    while ci < s:pcolor_max
       let bgColor = s:Strntok(g:pcolor_bg, ',', ci + 1)
       let fgColor = s:Strntok(g:pcolor_fg, ',', ci + 1)
+      let cterm_bgColor = s:Strntok(g:cterm_pcolor_bg, ',', ci + 1)
+      let cterm_fgColor = s:Strntok(g:cterm_pcolor_fg, ',', ci + 1)
      
       exec 'hi ' . s:pcolor_grp . ci .
-         \ ' guifg =' . fgColor . ' guibg=' . bgColor
+         \ ' ctermfg=' . cterm_fgColor . ' ctermbg=' . cterm_bgColor .
+         \ ' guifg=' . fgColor . ' guibg=' . bgColor
      
       let ci = ci + 1
    endw
