Index: .vim/bundle/interestingwords/plugin/interestingwords.vim
===================================================================
--- .vim.orig/bundle/interestingwords/plugin/interestingwords.vim
+++ .vim/bundle/interestingwords/plugin/interestingwords.vim
@@ -224,8 +224,8 @@ if !hasmapto('<Plug>InterestingWords')
     vnoremap <silent> <leader>k :call InterestingWords('v')<cr>
     nnoremap <silent> <leader>K :call UncolorAllWords()<cr>
 
-    nnoremap <silent> n :call WordNavigation(1)<cr>
-    nnoremap <silent> N :call WordNavigation(0)<cr>
+    " nnoremap <silent> n :call WordNavigation(1)<cr>
+    " nnoremap <silent> N :call WordNavigation(0)<cr>
 endif
 
 if g:interestingWordsDefaultMappings
