Index: vim/pack/vimscripts/opt/l9/autoload/l9/tempbuffer.vim
===================================================================
--- vim.orig/pack/vimscripts/opt/l9/autoload/l9/tempbuffer.vim
+++ vim/pack/vimscripts/opt/l9/autoload/l9/tempbuffer.vim
@@ -42,8 +42,22 @@ endfunction
 "           If less than 0, the window becomes full-screen. 
 " a:listener:
 "   a:listener.onClose(written)
-function l9#tempbuffer#openScratch(bufname, filetype, lines, topleft, vertical, height, listener)
-  let openCmdPrefix = (a:topleft ? 'topleft ' : '')
+function l9#tempbuffer#openScratch(bufname, filetype, lines, location, vertical, height, listener)
+  if a:location == ''
+      let l = ''
+  else
+      let l = 'topleft '
+  endif
+  "if a:location =~ '^top\(l\(e\(f\(t\)\?\)\?\)\?\)\?$'
+  "let l = 'topleft '
+  if a:location =~ '^lefta\(b\(a\(v\(e\)\?\)\?\)\?\)\?$' || a:location =~ '^abo\(o\(v\(e\(l\(e\(f\(t\)\?\)\?\)\?\)\?\)\?\)\?\)\?$'
+      let l = 'leftabove '
+  elseif a:location =~ '^rightb\(e\(l\(o\(w\)\?\)\?\)\?\)\?$' || a:location =~ '^bel\(o\(w\(r\(i\(g\(h\(t\)\?\)\?\)\?\)\?\)\?\)\?\)\?$'
+      let l = 'rightbelow '
+  elseif a:location =~ '^bot\(r\(i\(g\(h\(t\)\?\)\?\)\?\)\?\)\?$'
+      let l = 'botright '
+  endif
+  let openCmdPrefix = (l)
         \           . (a:vertical ? 'vertical ' : '')
         \           . (a:height > 0 ? a:height : '')
   if !exists('s:dataMap[a:bufname]') || !bufexists(s:dataMap[a:bufname].bufNr)
