Index: vim/pack/submodules/opt/lightline/autoload/lightline/colorscheme/PaperColor_light.vim
===================================================================
--- vim.orig/pack/submodules/opt/lightline/autoload/lightline/colorscheme/PaperColor_light.vim
+++ vim/pack/submodules/opt/lightline/autoload/lightline/colorscheme/PaperColor_light.vim
@@ -40,7 +40,7 @@ let s:p.normal.left = [ [ s:foreground,
 let s:p.normal.right = [ [ s:foreground, s:background ], [ s:statusline_active_fg, s:status ], [ s:statusline_active_fg, s:statusline_active_bg ] ]
 let s:p.normal.middle = [ [ s:statusline_active_fg, s:statusline_active_bg ]]
 let s:p.inactive.right = [ [ s:foreground, s:background ], [ s:foreground, s:background ] ]
-let s:p.inactive.left = [ [ s:foreground, s:background ], [ s:foreground, s:background ] ]
+let s:p.inactive.left = [ [ s:background, s:navy], [ s:foreground, s:background ] ]
 let s:p.inactive.middle = [ [ s:foreground, s:background ], ]
 let s:p.insert.left = [ [ s:blue, s:background ], [ s:statusline_active_fg, s:status ], [ s:statusline_active_fg, s:statusline_active_bg ] ]
 let s:p.replace.left = [ [ s:background, s:pink ], [s:statusline_active_fg, s:status ], [ s:statusline_active_fg, s:statusline_active_bg ] ]
