Index: .vim/addons/LiteTabPage/plugin/LiteTabPage.vim
===================================================================
--- .vim.orig/addons/LiteTabPage/plugin/LiteTabPage.vim	2010-10-22 12:39:20.517097184 +0800
+++ .vim/addons/LiteTabPage/plugin/LiteTabPage.vim	2010-10-22 12:39:26.217097185 +0800
@@ -30,7 +30,7 @@
 
 let loaded_litetabpage = 1
 
-com! -nargs=* -complete=file E tabnew <args>
+"com! -nargs=* -complete=file E tabnew <args>
 
 nnoremap <unique> <A-h> gT
 nnoremap <unique> <A-l> gt
