Index: .vim/bundle/LiteTabPage/plugin/LiteTabPage.vim
===================================================================
--- .vim.orig/bundle/LiteTabPage/plugin/LiteTabPage.vim
+++ .vim/bundle/LiteTabPage/plugin/LiteTabPage.vim
@@ -34,7 +34,7 @@ let loaded_litetabpage = 1
 
 set winaltkeys=no
 
-com! -nargs=* -complete=file E tabnew <args>
+"com! -nargs=* -complete=file E tabnew <args>
 
 nnoremap <unique> <A-1> 1gt
 nnoremap <unique> <A-2> 2gt
