Index: .vim/bundle/LiteTabPage/plugin/LiteTabPage.vim
===================================================================
--- .vim.orig/bundle/LiteTabPage/plugin/LiteTabPage.vim	2011-10-15 19:58:47.757848641 +0200
+++ .vim/bundle/LiteTabPage/plugin/LiteTabPage.vim	2011-10-15 20:00:36.190386765 +0200
@@ -51,6 +51,13 @@
 nnoremap <unique> <A-l> gt
 nnoremap <silent> <A-H> :call <SID>LiteTabMove(-2)<CR>
 nnoremap <silent> <A-L> :call <SID>LiteTabMove(0)<CR>
+nnoremap <silent> <unique> <A-j> :exec 'tabnext '.(exists('g:litetabpage_previous') ? g:litetabpage_previous : 1)<CR>
+
+let g:litetabpage_previous = 1
+augroup litetabpage
+	au!
+	au TabLeave * :let g:litetabpage_previous = tabpagenr()
+augroup END
 
 function! s:LiteTabMove(idx)
 	let index = tabpagenr() + a:idx
