Index: .vim/bundle/LiteTabPage/plugin/LiteTabPage.vim
===================================================================
--- .vim.orig/bundle/LiteTabPage/plugin/LiteTabPage.vim	2010-12-19 23:28:03.028510911 +0100
+++ .vim/bundle/LiteTabPage/plugin/LiteTabPage.vim	2010-12-19 23:29:41.043478910 +0100
@@ -34,7 +34,7 @@
 
 set winaltkeys=no
 
-com! -nargs=* -complete=file E tabnew <args>
+"com! -nargs=* -complete=file E tabnew <args>
 
 nnoremap <unique> <A-1> 1gt
 nnoremap <unique> <A-2> 2gt
