Index: .vim/colors/lucius.vim
===================================================================
--- .vim.orig/colors/lucius.vim
+++ .vim/colors/lucius.vim
@@ -492,7 +492,7 @@ hi TabLine      guifg=bg
 hi TabLineSel   guifg=fg
 hi WildMenu     guifg=fg
 if s:style == "light"
-    hi ColorColumn                  guibg=#d7d7af
+    hi ColorColumn                  guibg=#dadada
     hi CursorLineNr guifg=#9e9e9e   guibg=#dadada
     hi FoldColumn                   guibg=#bcbcbc
     hi Folded                       guibg=#bcbcbc
