Index: vim/.vim/plugin/lusty-explorer.vim
===================================================================
--- vim.orig/.vim/plugin/lusty-explorer.vim	2010-04-02 15:40:47.000000000 +0200
+++ vim/.vim/plugin/lusty-explorer.vim	2010-04-02 15:50:15.000000000 +0200
@@ -664,14 +664,16 @@
       end
 
       # Special characters
-      VIM::command "#{map} <Tab>    :call <SID>Lusty#{name}KeyPressed(9)<CR>"
+      VIM::command "#{map} <Tab>    :call <SID>Lusty#{name}KeyPressed(14)<CR>"
+      VIM::command "#{map} <S-Tab>    :call <SID>Lusty#{name}KeyPressed(16)<CR>"
       VIM::command "#{map} <Bslash> :call <SID>Lusty#{name}KeyPressed(92)<CR>"
       VIM::command "#{map} <Space>  :call <SID>Lusty#{name}KeyPressed(32)<CR>"
       VIM::command "#{map} \026|    :call <SID>Lusty#{name}KeyPressed(124)<CR>"
 
       VIM::command "#{map} <BS>     :call <SID>Lusty#{name}KeyPressed(8)<CR>"
       VIM::command "#{map} <Del>    :call <SID>Lusty#{name}KeyPressed(8)<CR>"
-      VIM::command "#{map} <C-h>    :call <SID>Lusty#{name}KeyPressed(8)<CR>"
+      VIM::command "#{map} <C-h>    :call <SID>Lusty#{name}KeyPressed(23)<CR>"
+      VIM::command "#{map} <C-BS>    :call <SID>Lusty#{name}KeyPressed(23)<CR>"
 
       VIM::command "#{map} <CR>     :call <SID>Lusty#{name}KeyPressed(13)<CR>"
       VIM::command "#{map} <S-CR>   :call <SID>Lusty#{name}KeyPressed(10)<CR>"
