Index: .vim/ftplugin/man.vim
===================================================================
--- .vim.orig/ftplugin/man.vim
+++ .vim/ftplugin/man.vim
@@ -159,10 +159,10 @@ func <SID>GetPage(...)
   silent exec "r!/usr/bin/man ".s:GetCmdArg(sect, page)." | col -b"
   " Remove blank lines from top and bottom.
   while getline(1) =~ '^\s*$'
-    silent keepj norm ggdd
+    1d
   endwhile
   while getline('$') =~ '^\s*$'
-    silent keepj norm Gdd
+    $d
   endwhile
   1
   setl ft=man nomod
