Index: vim/pack/ftplugins/start/man/ftplugin/man.vim
===================================================================
--- vim.orig/pack/ftplugins/start/man/ftplugin/man.vim
+++ vim/pack/ftplugins/start/man/ftplugin/man.vim
@@ -181,10 +181,10 @@ func <SID>GetPage(...)
   endif
   " Remove blank lines from top and bottom.
   while getline(1) =~ '^\s*$'
-    silent keepj norm ggdd
+    1d
   endwhile
   while getline('$') =~ '^\s*$'
-    silent keepj norm Gdd
+    $d
   endwhile
   1
   setl ft=man nomod
