Index: vimconfig/.vim/plugin/mark.vim
===================================================================
--- vimconfig.orig/.vim/plugin/mark.vim	2010-04-06 10:29:06.000000000 +0200
+++ vimconfig/.vim/plugin/mark.vim	2010-04-06 10:29:31.000000000 +0200
@@ -172,35 +172,35 @@
 if !hasmapto('<Plug>MarkSet', 'v')
 	vmap <unique> <silent> <Leader>m <Plug>MarkSet
 endif
-if !hasmapto('<Plug>MarkRegex', 'n')
-	nmap <unique> <silent> <Leader>r <Plug>MarkRegex
-endif
-if !hasmapto('<Plug>MarkRegex', 'v')
-	vmap <unique> <silent> <Leader>r <Plug>MarkRegex
-endif
-if !hasmapto('<Plug>MarkClear', 'n')
-	nmap <unique> <silent> <Leader>n <Plug>MarkClear
-endif
-" No default mapping for <Plug>MarkAllClear. 
-
-if !hasmapto('<Plug>MarkSearchCurrentNext', 'n')
-	nmap <unique> <silent> <Leader>* <Plug>MarkSearchCurrentNext
-endif
-if !hasmapto('<Plug>MarkSearchCurrentPrev', 'n')
-	nmap <unique> <silent> <Leader># <Plug>MarkSearchCurrentPrev
-endif
-if !hasmapto('<Plug>MarkSearchAnyNext', 'n')
-	nmap <unique> <silent> <Leader>/ <Plug>MarkSearchAnyNext
-endif
-if !hasmapto('<Plug>MarkSearchAnyPrev', 'n')
-	nmap <unique> <silent> <Leader>? <Plug>MarkSearchAnyPrev
-endif
-if !hasmapto('<Plug>MarkSearchNext', 'n')
-	nmap <unique> <silent> * <Plug>MarkSearchNext
-endif
-if !hasmapto('<Plug>MarkSearchPrev', 'n')
-	nmap <unique> <silent> # <Plug>MarkSearchPrev
-endif
+"if !hasmapto('<Plug>MarkRegex', 'n')
+"	nmap <unique> <silent> <Leader>r <Plug>MarkRegex
+"endif
+"if !hasmapto('<Plug>MarkRegex', 'v')
+"	vmap <unique> <silent> <Leader>r <Plug>MarkRegex
+"endif
+"if !hasmapto('<Plug>MarkClear', 'n')
+"	nmap <unique> <silent> <Leader>n <Plug>MarkClear
+"endif
+"" No default mapping for <Plug>MarkAllClear. 
+"
+"if !hasmapto('<Plug>MarkSearchCurrentNext', 'n')
+"	nmap <unique> <silent> <Leader>* <Plug>MarkSearchCurrentNext
+"endif
+"if !hasmapto('<Plug>MarkSearchCurrentPrev', 'n')
+"	nmap <unique> <silent> <Leader># <Plug>MarkSearchCurrentPrev
+"endif
+"if !hasmapto('<Plug>MarkSearchAnyNext', 'n')
+"	nmap <unique> <silent> <Leader>/ <Plug>MarkSearchAnyNext
+"endif
+"if !hasmapto('<Plug>MarkSearchAnyPrev', 'n')
+"	nmap <unique> <silent> <Leader>? <Plug>MarkSearchAnyPrev
+"endif
+"if !hasmapto('<Plug>MarkSearchNext', 'n')
+"	nmap <unique> <silent> * <Plug>MarkSearchNext
+"endif
+"if !hasmapto('<Plug>MarkSearchPrev', 'n')
+"	nmap <unique> <silent> # <Plug>MarkSearchPrev
+"endif
 
 
 "- commands -------------------------------------------------------------------
