Index: vim/.vim/plugin/mark.vim
===================================================================
--- vim.orig/.vim/plugin/mark.vim	2009-07-11 10:06:33.000000000 +0200
+++ vim/.vim/plugin/mark.vim	2009-07-11 10:08:10.000000000 +0200
@@ -173,29 +173,29 @@
 if !hasmapto('<Plug>MarkRegex', 'v')
 	vmap <unique> <silent> <Leader>r <Plug>MarkRegex
 endif
-if !hasmapto('<Plug>MarkClear', 'n')
-	nmap <unique> <silent> <Leader>n <Plug>MarkClear
-endif
+"if !hasmapto('<Plug>MarkClear', 'n')
+"	nmap <unique> <silent> <Leader>n <Plug>MarkClear
+"endif
 " No default mapping for <Plug>MarkAllClear. 
 
-if !hasmapto('<Plug>MarkSearchCurrentNext', 'n')
-	nmap <unique> <silent> <Leader>* <Plug>MarkSearchCurrentNext
-endif
-if !hasmapto('<Plug>MarkSearchCurrentPrev', 'n')
-	nmap <unique> <silent> <Leader># <Plug>MarkSearchCurrentPrev
-endif
-if !hasmapto('<Plug>MarkSearchAnyNext', 'n')
-	nmap <unique> <silent> <Leader>/ <Plug>MarkSearchAnyNext
-endif
-if !hasmapto('<Plug>MarkSearchAnyPrev', 'n')
-	nmap <unique> <silent> <Leader>? <Plug>MarkSearchAnyPrev
-endif
-if !hasmapto('<Plug>MarkSearchNext', 'n')
-	nmap <unique> <silent> * <Plug>MarkSearchNext
-endif
-if !hasmapto('<Plug>MarkSearchPrev', 'n')
-	nmap <unique> <silent> # <Plug>MarkSearchPrev
-endif
+"if !hasmapto('<Plug>MarkSearchCurrentNext', 'n')
+"	nmap <unique> <silent> <Leader>* <Plug>MarkSearchCurrentNext
+"endif
+"if !hasmapto('<Plug>MarkSearchCurrentPrev', 'n')
+"	nmap <unique> <silent> <Leader># <Plug>MarkSearchCurrentPrev
+"endif
+"if !hasmapto('<Plug>MarkSearchAnyNext', 'n')
+"	nmap <unique> <silent> <Leader>/ <Plug>MarkSearchAnyNext
+"endif
+"if !hasmapto('<Plug>MarkSearchAnyPrev', 'n')
+"	nmap <unique> <silent> <Leader>? <Plug>MarkSearchAnyPrev
+"endif
+"if !hasmapto('<Plug>MarkSearchNext', 'n')
+"	nmap <unique> <silent> * <Plug>MarkSearchNext
+"endif
+"if !hasmapto('<Plug>MarkSearchPrev', 'n')
+"	nmap <unique> <silent> # <Plug>MarkSearchPrev
+"endif
 
 
 "- commands -------------------------------------------------------------------
