Index: vim/.vim/plugin/NERD_commenter.vim
===================================================================
--- vim.orig/.vim/plugin/NERD_commenter.vim	2008-11-26 21:20:06.000000000 +0100
+++ vim/.vim/plugin/NERD_commenter.vim	2008-11-26 21:20:11.000000000 +0100
@@ -70,6 +70,7 @@
 call s:InitVariable("g:NERDShutUp", '0')
 call s:InitVariable("g:NERDSpaceDelims", 0)
 call s:InitVariable("g:NERDDelimiterRequests", 1)
+call s:InitVariable("g:NERDMapleader", ',c')
 
 
 
@@ -3291,18 +3292,18 @@
 endfunction
 
 if g:NERDCreateDefaultMappings
-    call s:CreateMaps('<plug>NERDCommenterComment',    ',cc')
-    call s:CreateMaps('<plug>NERDCommenterToggle',     ',c<space>')
-    call s:CreateMaps('<plug>NERDCommenterMinimal',    ',cm')
-    call s:CreateMaps('<plug>NERDCommenterSexy',       ',cs')
-    call s:CreateMaps('<plug>NERDCommenterInvert',     ',ci')
-    call s:CreateMaps('<plug>NERDCommenterYank',       ',cy')
-    call s:CreateMaps('<plug>NERDCommenterAlignLeft',  ',cl')
-    call s:CreateMaps('<plug>NERDCommenterAlignBoth',  ',cb')
-    call s:CreateMaps('<plug>NERDCommenterNest',       ',cn')
-    call s:CreateMaps('<plug>NERDCommenterUncomment',  ',cu')
-    call s:CreateMaps('<plug>NERDCommenterToEOL',      ',c$')
-    call s:CreateMaps('<plug>NERDCommenterAppend',     ',cA')
+    call s:CreateMaps('<plug>NERDCommenterComment',    NERDMapleader.'c')
+    call s:CreateMaps('<plug>NERDCommenterToggle',     NERDMapleader.'<space>')
+    call s:CreateMaps('<plug>NERDCommenterMinimal',    NERDMapleader.'m')
+    call s:CreateMaps('<plug>NERDCommenterSexy',       NERDMapleader.'s')
+    call s:CreateMaps('<plug>NERDCommenterInvert',     NERDMapleader.'i')
+    call s:CreateMaps('<plug>NERDCommenterYank',       NERDMapleader.'y')
+    call s:CreateMaps('<plug>NERDCommenterAlignLeft',  NERDMapleader.'l')
+    call s:CreateMaps('<plug>NERDCommenterAlignBoth',  NERDMapleader.'b')
+    call s:CreateMaps('<plug>NERDCommenterNest',       NERDMapleader.'n')
+    call s:CreateMaps('<plug>NERDCommenterUncomment',  NERDMapleader.'u')
+    call s:CreateMaps('<plug>NERDCommenterToEOL',      NERDMapleader.'$')
+    call s:CreateMaps('<plug>NERDCommenterAppend',     NERDMapleader.'A')
 
     if !hasmapto('<plug>NERDCommenterAltDelims', 'n')
         nmap ,ca <plug>NERDCommenterAltDelims
