Index: nvim/pack/submodules/opt/one/colors/one.vim
===================================================================
--- nvim.orig/pack/submodules/opt/one/colors/one.vim
+++ nvim/pack/submodules/opt/one/colors/one.vim
@@ -845,22 +845,22 @@ if has('gui_running') || has('termguicol
 
   " Neovim Terminal Colors --------------------------------------------------{{{
   if has('nvim')
-    let g:terminal_color_0  = "#353a44"
-    let g:terminal_color_8  = "#353a44"
-    let g:terminal_color_1  = "#e88388"
-    let g:terminal_color_9  = "#e88388"
-    let g:terminal_color_2  = "#a7cc8c"
-    let g:terminal_color_10 = "#a7cc8c"
-    let g:terminal_color_3  = "#ebca8d"
-    let g:terminal_color_11 = "#ebca8d"
-    let g:terminal_color_4  = "#72bef2"
-    let g:terminal_color_12 = "#72bef2"
-    let g:terminal_color_5  = "#d291e4"
-    let g:terminal_color_13 = "#d291e4"
-    let g:terminal_color_6  = "#65c2cd"
-    let g:terminal_color_14 = "#65c2cd"
-    let g:terminal_color_7  = "#e3e5e9"
-    let g:terminal_color_15 = "#e3e5e9"
+    let g:terminal_color_0  = "#282c34"
+    let g:terminal_color_1  = "#e06c75"
+    let g:terminal_color_2  = "#98c379"
+    let g:terminal_color_3  = "#d19a66"
+    let g:terminal_color_4  = "#61afef"
+    let g:terminal_color_5  = "#c678dd"
+    let g:terminal_color_6  = "#56b6c2"
+    let g:terminal_color_7  = "#abb2bf"
+    let g:terminal_color_8  = "#5c6370"
+    let g:terminal_color_9  = "#e06c75"
+    let g:terminal_color_10 = "#98c379"
+    let g:terminal_color_11 = "#d19a66"
+    let g:terminal_color_12 = "#61afef"
+    let g:terminal_color_13 = "#c678dd"
+    let g:terminal_color_14 = "#56b6c2"
+    let g:terminal_color_15 = "#ffffff"
   endif
 
   " Delete functions =========================================================={{{
