Index: .vim/bundle/orgmode/ftplugin/org.vim
===================================================================
--- .vim.orig/bundle/orgmode/ftplugin/org.vim
+++ .vim/bundle/orgmode/ftplugin/org.vim
@@ -27,6 +27,8 @@ if &cp || exists("g:loaded_org")
 endif
 let g:loaded_org = 1
 
+IP speeddating utl
+
 " general setting plugins that should be loaded and their order
 if ! exists('g:org_plugins') && ! exists('b:org_plugins')
 	let g:org_plugins = ['ShowHide', '|', 'Navigator', 'EditStructure', '|', 'Hyperlinks', '|', 'Todo', 'TagsProperties', 'Date', 'Agenda', 'Misc', '|', 'Export']
