Index: vim/bundle/orgmode/ftplugin/org.vim
===================================================================
--- vim.orig/bundle/orgmode/ftplugin/org.vim
+++ vim/bundle/orgmode/ftplugin/org.vim
@@ -30,6 +30,8 @@ if &cp || exists("g:loaded_org")
 endif
 let g:loaded_org = 1
 
+IP speeddating utl
+
 " Default org plugins that will be loaded (in the given order)
 if ! exists('g:org_plugins') && ! exists('b:org_plugins')
 	let g:org_plugins = ['ShowHide', '|', 'Navigator', 'EditStructure', '|', 'Hyperlinks', '|', 'Todo', 'TagsProperties', 'Date', 'Agenda', 'Misc', '|', 'Export', 'EditCheckbox']
