Index: vim/pack/ftplugins/start/orgmode/ftplugin/org.vim
===================================================================
--- vim.orig/pack/ftplugins/start/orgmode/ftplugin/org.vim
+++ vim/pack/ftplugins/start/orgmode/ftplugin/org.vim
@@ -49,6 +49,8 @@ if ! exists('b:did_ftplugin')
 	endif
 endif
 
+silent! IP speeddating utl
+
 " Load orgmode just once {{{1
 if &cp || exists("g:loaded_org")
     finish
