Index: .vim/pack/submodules/opt/papercolor-theme/colors/PaperColor.vim
===================================================================
--- .vim.orig/pack/submodules/opt/papercolor-theme/colors/PaperColor.vim
+++ .vim/pack/submodules/opt/papercolor-theme/colors/PaperColor.vim
@@ -34,7 +34,7 @@ let s:themes['default'].light = {
       \       'color01' : ['#af0000', '124'],
       \       'color02' : ['#008700', '28'],
       \       'color03' : ['#5f8700', '64'],
-      \       'color04' : ['#0087af', '31'],
+      \       'color04' : ['#87afd7', '110'],
       \       'color05' : ['#878787', '102'],
       \       'color06' : ['#005f87', '24'],
       \       'color07' : ['#444444', '238'],
@@ -46,7 +46,7 @@ let s:themes['default'].light = {
       \       'color13' : ['#d75f00', '166'],
       \       'color14' : ['#005faf', '25'],
       \       'color15' : ['#005f87', '24'],
-      \       'color16' : ['#0087af', '31'],
+      \       'color16' : ['#87afd7', '110'],
       \       'color17' : ['#008700', '28'],
       \       'cursor_fg' : ['#eeeeee', '255'],
       \       'cursor_bg' : ['#005f87', '24'],
@@ -62,8 +62,8 @@ let s:themes['default'].light = {
       \       'linenumber_bg' : ['#eeeeee', '255'],
       \       'vertsplit_fg' : ['#005f87', '24'],
       \       'vertsplit_bg' : ['#eeeeee', '255'],
-      \       'statusline_active_fg' : ['#e4e4e4', '254'],
-      \       'statusline_active_bg' : ['#005f87', '24'],
+      \       'statusline_active_fg' : ['#444444', '238'],
+      \       'statusline_active_bg' : ['#eeeeee', '255'],
       \       'statusline_inactive_fg' : ['#444444', '238'],
       \       'statusline_inactive_bg' : ['#d0d0d0', '252'],
       \       'todo_fg' : ['#00af5f', '35'],
@@ -73,7 +73,7 @@ let s:themes['default'].light = {
       \       'matchparen_bg' : ['#c6c6c6', '251'],
       \       'matchparen_fg' : ['#005f87', '24'],
       \       'visual_fg' : ['#eeeeee', '255'],
-      \       'visual_bg' : ['#0087af', '31'],
+      \       'visual_bg' : ['#87afd7', '110'],
       \       'folded_fg' : ['#005f87', '24'],
       \       'folded_bg' : ['#afd7ff', '153'],
       \       'wildmenu_fg': ['#444444', '238'],
@@ -86,7 +86,7 @@ let s:themes['default'].light = {
       \       'diffadd_bg':    ['#afffaf', '157'],
       \       'diffdelete_fg': ['#af0000', '124'],
       \       'diffdelete_bg': ['#ffd7ff', '225'],
-      \       'difftext_fg':   ['#0087af', '31'],
+      \       'difftext_fg':   ['#87afd7', '110'],
       \       'difftext_bg':   ['#ffffd7', '230'],
       \       'diffchange_fg': ['#444444', '238'],
       \       'diffchange_bg': ['#ffffaf', '229'],
@@ -94,14 +94,14 @@ let s:themes['default'].light = {
       \       'tabline_active_fg':   ['#444444', '238'],
       \       'tabline_active_bg':   ['#e4e4e4', '254'],
       \       'tabline_inactive_fg': ['#eeeeee', '255'],
-      \       'tabline_inactive_bg': ['#0087af', '31'],
+      \       'tabline_inactive_bg': ['#87afd7', '110'],
       \       'buftabline_bg':          ['#005f87', '24'],
       \       'buftabline_current_fg':  ['#444444', '238'],
       \       'buftabline_current_bg':  ['#e4e4e4', '254'],
       \       'buftabline_active_fg': ['#eeeeee', '255'],
       \       'buftabline_active_bg': ['#005faf', '25'],
       \       'buftabline_inactive_fg': ['#eeeeee', '255'],
-      \       'buftabline_inactive_bg': ['#0087af', '31']
+      \       'buftabline_inactive_bg': ['#87afd7', '110']
       \     }
       \   }
 
@@ -771,7 +771,7 @@ fun! s:set_highlightings_variable()
   call s:HL("LineNr", s:linenumber_fg, s:linenumber_bg, "")
   call s:HL("Conceal", s:linenumber_fg, s:linenumber_bg, "")
 
-  call s:HL("StatusLine", s:statusline_active_bg, s:statusline_active_fg, "")
+  call s:HL("StatusLine", s:statusline_active_bg, s:statusline_active_fg, "NONE,reverse")
   call s:HL("StatusLineNC", s:statusline_inactive_bg, s:statusline_inactive_fg, "")
 
   call s:HL("VertSplit", s:vertsplit_bg, s:vertsplit_fg, "")
