Index: .vim/bundle/papercolor-theme/colors/PaperColor.vim
===================================================================
--- .vim.orig/bundle/papercolor-theme/colors/PaperColor.vim
+++ .vim/bundle/papercolor-theme/colors/PaperColor.vim
@@ -63,9 +63,9 @@ let s:themes['default'].light = {
       \       'vertsplit_fg' : ['#005f87', '24'],
       \       'vertsplit_bg' : ['#eeeeee', '255'],
       \       'statusline_active_fg' : ['#e4e4e4', '254'],
-      \       'statusline_active_bg' : ['#005f87', '24'],
+      \       'statusline_active_bg' : ['#eeeeee', '255'],
       \       'statusline_inactive_fg' : ['#444444', '238'],
-      \       'statusline_inactive_bg' : ['#d0d0d0', '252'],
+      \       'statusline_inactive_bg' : ['#eeeeee', '255'],
       \       'todo_fg' : ['#00af5f', '35'],
       \       'todo_bg' : ['#eeeeee', '255'],
       \       'error_fg' : ['#af0000', '124'],
