Index: vim/pack/submodules/opt/papercolor-theme/colors/PaperColor.vim
===================================================================
--- vim.orig/pack/submodules/opt/papercolor-theme/colors/PaperColor.vim
+++ vim/pack/submodules/opt/papercolor-theme/colors/PaperColor.vim
@@ -15,6 +15,11 @@ let s:version = '0.9.x'
 " - The first section right after this note is where themes are defined. Theme
 "   designers only need to work on this section.
 
+if exists('g:papercolor_second_initialization')
+    finish
+endif
+let g:papercolor_second_initialization = 1
+
 " THEMES: {{{
 
 let s:themes = {}
