Index: .vim/bundle/papercolor-theme/colors/PaperColor.vim
===================================================================
--- .vim.orig/bundle/papercolor-theme/colors/PaperColor.vim
+++ .vim/bundle/papercolor-theme/colors/PaperColor.vim
@@ -373,7 +373,7 @@ else " LIGHT VARIANT
   let s:statusline_active_fg   = s:window
   let s:statusline_active_bg   = s:navy
   let s:statusline_inactive_fg = s:foreground
-  let s:statusline_inactive_bg = ['#d0d0d0', '252']
+  let s:statusline_inactive_bg = s:background
 
   " Search:
   let s:search_fg = s:foreground
