Index: .vim/bundle/pep8/ftplugin/python_pep8.vim
===================================================================
--- .vim.orig/bundle/pep8/ftplugin/python_pep8.vim	2012-05-04 15:25:26.000000000 +0200
+++ .vim/bundle/pep8/ftplugin/python_pep8.vim	2012-05-04 15:25:33.000000000 +0200
@@ -33,8 +33,12 @@
         endif
 
     " perform the grep itself
+    let options=""
+    if exists('g:pep8_options')
+		let options=g:pep8_options
+	endif
     let &grepformat="%f:%l:%c: %m"
-    let &grepprg=s:pep8_cmd . " --repeat"
+    let &grepprg=s:pep8_cmd . " " . options . " --repeat"
     silent! grep! %
 
         " restore grep settings
