Index: .vim/bundle/powerline/autoload/Powerline/Themes/default.vim
===================================================================
--- .vim.orig/bundle/powerline/autoload/Powerline/Themes/default.vim
+++ .vim/bundle/powerline/autoload/Powerline/Themes/default.vim
@@ -5,6 +5,7 @@ let g:Powerline#Themes#default#theme = P
 		\ , 'fileinfo'
 		\ , 'syntastic:errors'
 		\ , Pl#Segment#Truncate()
+		\ , 'warnings'
 		\ , 'cfi:current_function'
 		\ , Pl#Segment#Split()
 		\ , 'rvm:string'
Index: .vim/bundle/powerline/autoload/Powerline/Colorschemes/default.vim
===================================================================
--- .vim.orig/bundle/powerline/autoload/Powerline/Colorschemes/default.vim
+++ .vim/bundle/powerline/autoload/Powerline/Colorschemes/default.vim
@@ -66,6 +66,12 @@ let g:Powerline#Colorschemes#default#col
 		\ 'i': ['white', 'darkblue', ['bold']],
 		\ }),
 	\
+	\ Pl#Hi#Segments(['fileinfo.alternate'], {
+		\ 'n': ['gray8', 'gray4', ['bold']],
+		\ 'N': ['gray4', 'gray1', ['bold']],
+		\ 'i': ['white', 'darkblue', ['bold']],
+		\ }),
+	\
 	\ Pl#Hi#Segments(['static_str'], {
 		\ 'n': ['white', 'gray4'],
 		\ 'N': ['gray7', 'gray1'],
@@ -76,6 +82,12 @@ let g:Powerline#Colorschemes#default#col
 		\ 'n': ['brightestred', ['bold']],
 		\ 'N': ['darkred'],
 		\ 'i': ['brightestred', ['bold']],
+		\ }),
+	\
+	\ Pl#Hi#Segments(['warnings'], {
+		\ 'n': ['brightestred', ['bold']],
+		\ 'N': ['darkred'],
+		\ 'i': ['brightestred', ['bold']],
 		\ }),
 	\
 	\ Pl#Hi#Segments(['current_function', 'fileformat', 'fileencoding', 'pwd', 'filetype', 'rvm:string', 'rvm:statusline', 'virtualenv:statusline', 'charcode', 'currhigroup'], {
Index: .vim/bundle/powerline/autoload/Powerline/Segments.vim
===================================================================
--- .vim.orig/bundle/powerline/autoload/Powerline/Segments.vim
+++ .vim/bundle/powerline/autoload/Powerline/Segments.vim
@@ -8,6 +8,12 @@ let g:Powerline#Segments#segments = Pl#S
 		\ Pl#Segment#Create('name'        , '%t'),
 		\ Pl#Segment#Create('flags.mod'   , '%M'),
 		\ Pl#Segment#Create('flags.type'  , '%H%W'),
+		\ Pl#Segment#Create('alternate'   , '%{expand("#:t")}'),
+	\ ),
+	\ Pl#Segment#Create('warnings',
+		\ Pl#Segment#Create('bomb'        , '%{(&bomb  == 1 ? "BOM(B)" : "")}'),
+		\ Pl#Segment#Create('paste'       , '%{(&paste == 1 ? "PASTE"  : "")}'),
+		\ Pl#Segment#Create('paste'       , '%{(&eol   == 0 ? "NOEOL"  : "")}'),
 	\ ),
 	\ Pl#Segment#Create('filename'        , '%t'),
 	\ Pl#Segment#Create('filesize'        , '%{Powerline#Functions#GetFilesize()}', Pl#Segment#Modes('!N')),
