Index: .vim/bundle/powerline/autoload/Powerline/Themes/default.vim
===================================================================
--- .vim.orig/bundle/powerline/autoload/Powerline/Themes/default.vim	2012-04-11 18:09:07.000000000 +0200
+++ .vim/bundle/powerline/autoload/Powerline/Themes/default.vim	2012-04-11 18:09:11.000000000 +0200
@@ -5,6 +5,7 @@
 		\ , 'fileinfo'
 		\ , 'syntastic:errors'
 		\ , Pl#Segment#Truncate()
+		\ , 'warnings'
 		\ , 'cfi:current_function'
 		\ , Pl#Segment#Split()
 		\ , 'rvm:string'
Index: .vim/bundle/powerline/autoload/Powerline/Colorschemes/default.vim
===================================================================
--- .vim.orig/bundle/powerline/autoload/Powerline/Colorschemes/default.vim	2012-04-11 18:09:07.000000000 +0200
+++ .vim/bundle/powerline/autoload/Powerline/Colorschemes/default.vim	2012-04-11 18:14:46.000000000 +0200
@@ -66,6 +66,12 @@
 		\ 'i': ['white', 'darkblue', ['bold']],
 		\ }),
 	\
+	\ Pl#Hi#Segments(['fileinfo.alternate'], {
+		\ 'n': ['gray8', 'gray4', ['bold']],
+		\ 'N': ['gray4', 'gray1', ['bold']],
+		\ 'i': ['white', 'darkblue', ['bold']],
+		\ }),
+	\
 	\ Pl#Hi#Segments(['static_str'], {
 		\ 'n': ['white', 'gray4'],
 		\ 'N': ['gray7', 'gray1'],
@@ -78,6 +84,18 @@
 		\ 'i': ['brightestred', ['bold']],
 		\ }),
 	\
+	\ Pl#Hi#Segments(['warnings.bomb', 'warnings.paste',  'warnings.neol'], {
+		\ 'n': ['brightestred', ['bold']],
+		\ 'N': ['darkred'],
+		\ 'i': ['brightestred', ['bold']],
+		\ }),
+	\
+	\ Pl#Hi#Segments(['warnings'], {
+		\ 'n': ['gray8', 'gray2'],
+		\ 'N': ['white', 'gray0'],
+		\ 'i': ['mediumcyan', 'darkestblue'],
+		\ }),
+	\
 	\ Pl#Hi#Segments(['current_function', 'fileformat', 'fileencoding', 'pwd', 'filetype', 'rvm:string', 'rvm:statusline', 'virtualenv:statusline', 'charcode', 'currhigroup'], {
 		\ 'n': ['gray8', 'gray2'],
 		\ 'i': ['mediumcyan', 'darkestblue'],
Index: .vim/bundle/powerline/autoload/Powerline/Segments.vim
===================================================================
--- .vim.orig/bundle/powerline/autoload/Powerline/Segments.vim	2012-04-11 18:09:07.000000000 +0200
+++ .vim/bundle/powerline/autoload/Powerline/Segments.vim	2012-04-11 18:22:16.000000000 +0200
@@ -8,6 +8,12 @@
 		\ Pl#Segment#Create('name'        , '%t'),
 		\ Pl#Segment#Create('flags.mod'   , '%M'),
 		\ Pl#Segment#Create('flags.type'  , '%H%W'),
+		\ Pl#Segment#Create('alternate'   , '%{expand("#:t")}', Pl#Segment#Modes('!N')),
+	\ ),
+	\ Pl#Segment#Create('warnings',
+		\ Pl#Segment#Create('bomb'        , '%{(&bomb  == 1 ? "BOM(B)" : "")}', Pl#Segment#Modes('!N')),
+		\ Pl#Segment#Create('paste'       , '%{(&paste == 1 ? "PASTE"  : "")}', Pl#Segment#Modes('!N')),
+		\ Pl#Segment#Create('neol'        , '%{(&eol   == 0 ? "NOEOL"  : "")}', Pl#Segment#Modes('!N')),
 	\ ),
 	\ Pl#Segment#Create('filename'        , '%t'),
 	\ Pl#Segment#Create('filesize'        , '%{Powerline#Functions#GetFilesize()}', Pl#Segment#Modes('!N')),
