Index: .vim/ipi/pydoc910/plugin/pydoc.vim
===================================================================
--- .vim.orig/ipi/pydoc910/plugin/pydoc.vim	2010-12-19 23:28:02.840604911 +0100
+++ .vim/ipi/pydoc910/plugin/pydoc.vim	2010-12-19 23:30:03.084452909 +0100
@@ -28,7 +28,7 @@
 "Please feel free to contact me.
 
 
-set switchbuf=useopen
+"set switchbuf=useopen
 function! ShowPyDoc(name, type)
     if !exists('g:pydoc_cmd')
         let g:pydoc_cmd = 'pydoc'
