Index: .vim/bundle/pythonhelper/plugin/pythonhelper.vim
===================================================================
--- .vim.orig/bundle/pythonhelper/plugin/pythonhelper.vim	2010-12-19 23:28:02.224912911 +0100
+++ .vim/bundle/pythonhelper/plugin/pythonhelper.vim	2010-12-19 23:30:12.387798911 +0100
@@ -752,7 +752,7 @@
 " the status line will be displayed for every window
 set laststatus=2
 " set the status line to display some useful information
-set stl=%-f%r\ %2*%m%*\ \ \ \ %1*%{TagInStatusLine()}%*%=[%l:%c]\ \ \ \ [buf\ %n]
+"set stl=%-f%r\ %2*%m%*\ \ \ \ %1*%{TagInStatusLine()}%*%=[%l:%c]\ \ \ \ [buf\ %n]
 
 " }}}
 
