Index: .vim/bundle/pythonhelper/plugin/pythonhelper.vim
===================================================================
--- .vim.orig/bundle/pythonhelper/plugin/pythonhelper.vim	2010-12-19 23:31:18.466742910 +0100
+++ .vim/bundle/pythonhelper/plugin/pythonhelper.vim	2010-12-19 23:31:52.537698910 +0100
@@ -25,6 +25,13 @@
 " 2. Copy script pythonhelper.vim to the $HOME/.vim/plugin directory
 " 3. Run Vim and open any python file.
 " 
+
+if exists('loaded_pythonhelper') || ! has ('python')
+    finish
+endif
+
+let loaded_pythonhelper = 1
+
 python << EOS
 
 # import of required modules {{{
