Index: .vim/bundle/python_fn/ftplugin/python_fn.vim
===================================================================
--- .vim.orig/bundle/python_fn/ftplugin/python_fn.vim	2011-02-28 12:45:38.000000000 +0100
+++ .vim/bundle/python_fn/ftplugin/python_fn.vim	2011-02-28 12:46:09.000000000 +0100
@@ -27,7 +27,7 @@
 " vim (>= 7)
 "
 " Shortcuts:
-"   ]t      -- Jump to beginning of block
+"   [t      -- Jump to beginning of block
 "   ]e      -- Jump to end of block
 "   ]v      -- Select (Visual Line Mode) block
 "   ]<      -- Shift block to left
@@ -45,15 +45,15 @@
 endif
 let b:loaded_py_ftplugin = 1
 
-map  ]t   :PBoB<CR>
-vmap ]t   :<C-U>PBOB<CR>m'gv``
+map  [t   :PBoB<CR>
+vmap [t   :<C-U>PBOB<CR>m'gv``
 map  ]e   :PEoB<CR>
 vmap ]e   :<C-U>PEoB<CR>m'gv``
 
-map  ]v   ]tV]e
-map  ]<   ]tV]e<
+map  ]v   [tV]e
+map  ]<   [tV]e<
 vmap ]<   <
-map  ]>   ]tV]e>
+map  ]>   [tV]e>
 vmap ]>   >
 
 map  ]#   :call PythonCommentSelection()<CR>
@@ -91,7 +91,7 @@
     \:call UpdateMenu()<CR>
 nmenu &Python.-Sep1- :
 nmenu <silent> &Python.Beginning\ of\ Block<Tab>[t 
-    \]t
+    \[t
 nmenu <silent> &Python.End\ of\ Block<Tab>]e 
     \]e
 nmenu &Python.-Sep2- :
