Index: vimconfig/.vim/addons/qfn/plugin/qfn.vim
===================================================================
--- vimconfig.orig/.vim/addons/qfn/plugin/qfn.vim	2009-10-03 16:10:29.000000000 +0200
+++ vimconfig/.vim/addons/qfn/plugin/qfn.vim	2009-10-03 16:14:02.000000000 +0200
@@ -30,7 +30,7 @@
   endif
   let entry = {}
   " may need expand() here.
-  let entry["filename"] = bufname("%")
+  let entry["filename"] = expand("%:p")
   let entry["lnum"] = line('.')
   let entry["col"] = col('.')
   let entry["vcol"] = ''
