Index: vim/.vim/plugin/qfn.vim
===================================================================
--- vim.orig/.vim/plugin/qfn.vim	2009-09-11 22:13:27.000000000 +0200
+++ vim/.vim/plugin/qfn.vim	2009-09-11 22:14:07.000000000 +0200
@@ -30,7 +30,7 @@
   endif
   let entry = {}
   " may need expand() here.
-  let entry["filename"] = bufname("%")
+  let entry["filename"] = bufname("%:p")
   let entry["lnum"] = line('.')
   let entry["col"] = col('.')
   let entry["vcol"] = ''
