Index: .vim/bundle/qfn/plugin/qfn.vim
===================================================================
--- .vim.orig/bundle/qfn/plugin/qfn.vim	2010-12-19 23:30:05.959014911 +0100
+++ .vim/bundle/qfn/plugin/qfn.vim	2010-12-19 23:30:06.906540911 +0100
@@ -83,11 +83,11 @@
 "endfunction
 
 " basic maps
-if !hasmapto('<Plug>QuickFixNote')
-  map <unique> <Leader>m <Plug>QuickFixNote
-endif
-
-if !hasmapto('<Plug>QuickFixSave')
-  map <unique> <Leader>s <Plug>QuickFixSave quickfix.err
-endif
+"if !hasmapto('<Plug>QuickFixNote')
+"  map <unique> <Leader>m <Plug>QuickFixNote
+"endif
+"
+"if !hasmapto('<Plug>QuickFixSave')
+"  map <unique> <Leader>s <Plug>QuickFixSave quickfix.err
+"endif
 
