Index: .vim/bundle/qfn/plugin/qfn.vim
===================================================================
--- .vim.orig/bundle/qfn/plugin/qfn.vim	2010-12-19 23:28:02.716666911 +0100
+++ .vim/bundle/qfn/plugin/qfn.vim	2010-12-19 23:30:05.959014911 +0100
@@ -30,7 +30,7 @@
   endif
   let entry = {}
   " may need expand() here.
-  let entry["filename"] = bufname("%")
+  let entry["filename"] = expand("%:p")
   let entry["lnum"] = line('.')
   let entry["col"] = col('.')
   let entry["vcol"] = ''
