Index: vim/pack/submodules/start/ranger/plugin/ranger.vim
===================================================================
--- vim.orig/pack/submodules/start/ranger/plugin/ranger.vim
+++ vim/pack/submodules/start/ranger/plugin/ranger.vim
@@ -62,15 +62,27 @@ function! s:RangerChooserForNeoVim(dirna
     startinsert
 endfunction
 
-function! s:RangerChooser(dirname)
-    if isdirectory(a:dirname)
+function! s:RangerChooser(bang, ...)
+    if a:0 > 0
+        let dir = a:1
+    else
+        let dir = getcwd()
+    endif
+    if a:bang == '!'
+        let dir = expand('%:p:h')
+    endif
+    if isdirectory(dir)
         if has('nvim')
-            call s:RangerChooserForNeoVim(a:dirname)
+            call s:RangerChooserForNeoVim(dir)
         else
-            call s:RangerChooserForAncientVim(a:dirname)
+            call s:RangerChooserForAncientVim(dir)
         endif
     endif
 endfunction
 
-au BufEnter * silent call s:RangerChooser(expand("<amatch>"))
-let g:loaded_netrwPlugin = 'disable'
+" au BufEnter * silent call s:RangerChooser('', expand("<amatch>"))
+" let g:loaded_netrwPlugin = 'disable'
+
+command! -nargs=? -bar -bang RangerChooser :call s:RangerChooser("<bang>", <f-args>)
+nnoremap <leader>e :<C-U>RangerChooser<CR>
+nnoremap <leader>E :<C-U>RangerChooser!<CR>
