Index: nvim/lazy/vim-rsi/plugin/rsi.vim
===================================================================
--- nvim.orig/lazy/vim-rsi/plugin/rsi.vim
+++ nvim/lazy/vim-rsi/plugin/rsi.vim
@@ -14,9 +14,9 @@ if &ttimeoutlen == -1
 endif
 
 inoremap        <C-A> <C-O>^
-inoremap   <C-X><C-A> <C-A>
+" inoremap   <C-X><C-A> <C-A>
 cnoremap        <C-A> <Home>
-cnoremap   <C-X><C-A> <C-A>
+" cnoremap   <C-X><C-A> <C-A>
 
 inoremap <expr> <C-B> getline('.')=~'^\s*$'&&col('.')>strlen(getline('.'))?"0\<Lt>C-D>\<Lt>Esc>kJs":"\<Lt>Left>"
 cnoremap        <C-B> <Left>
