Index: vim/bundle/shootingstar/plugin/shootingstar.vim
===================================================================
--- vim.orig/bundle/shootingstar/plugin/shootingstar.vim
+++ vim/bundle/shootingstar/plugin/shootingstar.vim
@@ -26,5 +26,5 @@ function! <SID>Shootingstar()
     endif
 endfunction
 
-nnoremap <Leader>* /\M\<<C-r>=<SID>Shootingstar()<CR>\><CR>
-nnoremap <Leader># ?\M\<<C-r>=<SID>Shootingstar()<CR>\><CR>
+nnoremap <Leader>* msHmt`/\M\<<C-r>=<SID>Shootingstar()<CR>\><CR>'tzt`s
+nnoremap <Leader># msHmt`?\M\<<C-r>=<SID>Shootingstar()<CR>\><CR>'tzt`s
