Index: .vim/bundle/shootingstar/plugin/shootingstar.vim
===================================================================
--- .vim.orig/bundle/shootingstar/plugin/shootingstar.vim
+++ .vim/bundle/shootingstar/plugin/shootingstar.vim
@@ -26,7 +26,7 @@ function! <SID>Shootingstar()
     endif
 endfunction
 
-nnoremap <Leader>* /\M\<<C-r>=<SID>Shootingstar()<CR>\><CR>
-nnoremap <Leader># ?\M\<<C-r>=<SID>Shootingstar()<CR>\><CR>
-nnoremap <Leader>g* /\M<C-r>=<SID>Shootingstar()<CR><CR>
-nnoremap <Leader>g# ?\M<C-r>=<SID>Shootingstar()<CR><CR>
+nnoremap <Leader>* msHmt`s/\M\<<C-r>=<SID>Shootingstar()<CR>\><CR>'tzt`s
+nnoremap <Leader># msHmt`s?\M\<<C-r>=<SID>Shootingstar()<CR>\><CR>'tzt`s
+nnoremap <Leader>g* msHmt`s/\M<C-r>=<SID>Shootingstar()<CR><CR>'tzt`s
+nnoremap <Leader>g# msHmt`s?\M<C-r>=<SID>Shootingstar()<CR><CR>'tzt`s
