Index: vim/.vim/snippets/_.snippets
===================================================================
--- vim.orig/.vim/snippets/_.snippets	2010-01-22 08:21:50.685903080 +0100
+++ vim/.vim/snippets/_.snippets	2010-01-22 08:22:02.445907356 +0100
@@ -5,3 +5,106 @@
 	`&enc[:2] == "utf" ? "©" : "(c)"` Copyright `strftime("%Y")` ${1:`g:snips_author`}. All Rights Reserved.${2}
 snippet date
 	`strftime("%Y-%m-%d")`
+snippet http
+	http://${1:www.}${2}${3:.de}/${4}
+snippet dt
+	`strftime("%F %R")`
+snippet dl
+	`strftime("%d. %B %Y")`
+snippet rfcd
+	`strftime("%a, %d %b %Y %H:%M:%S %z")`
+snippet time
+	`strftime("%R")`
+snippet vi
+	`printf("vi: ft=%s:tw=%s:sw=%s:ts=%s", &filetype, &textwidth, &shiftwidth, &tabstop)`
+snippet www
+	`g:userWWW`
+snippet email
+	`g:userEmail`
+snippet phone
+	`g:userPhone`
+snippet addr
+	`g:userAddr`
+snippet city
+	`g:userCity`
+snippet zip
+	`g:userZip`
+snippet street
+	`g:userStreet`
+snippet fn
+	`g:userGivenName` `g:userName`
+snippet n
+	`g:userGivenName`
+snippet sgd
+	Sehr geehrte Damen und Herren,
+
+	${1}
+snippet sgh
+	Sehr geehrter Herr ${1},
+
+	${2}
+snippet sgf
+	Sehr geehrte Frau ${1},
+
+	${2}
+snippet lg
+	Liebe Grüße,
+
+	${1}
+snippet mfg
+	Mit freundlichen Grüßen,
+
+	${1}
+snippet g
+	Grüße,
+
+	${1}
+snippet vd
+	Vielen Dank,
+
+	${1}
+snippet G
+	Greetings,
+
+	${1}
+snippet kr
+	Kind Regards,
+
+	${1}
+snippet bug
+	Bug #
+snippet tick
+	Ticket #
+snippet ticket
+	Ticket #
+snippet gpl
+	# ${1:`Filename()`}
+	#  ${2:description}
+	#
+	# Depends: ${3:dependencies}
+	#
+	# Copyright (C) `strftime("%Y")` ${4:Copyright holder}
+	#
+	# http://www.${5:website}/
+	#
+	# All rights reserved.
+	#
+	# This program is free software; you can redistribute it and/or modify
+	# it under the terms of the GNU General Public License version 2 as
+	# published by the Free Software Foundation.
+	#
+	# Binary versions of this file provided by Univention to you as
+	# well as other copyrighted, protected or trademarked materials like
+	# Logos, graphics, fonts, specific documentations and configurations,
+	# cryptographic keys etc. are subject to a license agreement between
+	# you and Univention.
+	#
+	# This program is distributed in the hope that it will be useful,
+	# but WITHOUT ANY WARRANTY; without even the implied warranty of
+	# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	# GNU General Public License for more details.
+	#
+	# You should have received a copy of the GNU General Public License
+	# along with this program; if not, write to the Free Software
+	# Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+	${6}
Index: vim/.vim/snippets/tex.snippets
===================================================================
--- vim.orig/.vim/snippets/tex.snippets	2010-01-22 08:22:09.325905717 +0100
+++ vim/.vim/snippets/tex.snippets	2010-01-22 08:22:13.225905020 +0100
@@ -113,3 +113,60 @@
 	${1:Section}~\ref{${2:sec:}}${3}
 snippet page
 	${1:page}~\pageref{${2}}${3}
+snippet letter
+	%!/usr/bin/env latex
+	\documentclass[a4paper,12pt]{dinbrief}
+	\pagestyle{empty}
+
+	\usepackage{ucs}
+	\usepackage[utf8x]{inputenc}
+	\usepackage[T1]{fontenc}              % Verwenden von DC-Fonts
+	\usepackage{ngerman}
+	%\usepackage[pdftitle={<+TITLE+>},colorlinks=true]{hyperref}
+	\usepackage[pdftitle={${1:subject}}]{hyperref}
+	\PrerenderUnicode{ä}
+
+	\begin{document}
+	%\setaddressllhpos{100truemm}
+	%\setaddressllvpos{96truemm}
+	%\setaddressllcorner{20truemm}{72truemm}
+
+	\stdaddress{${2:author}\\ ${3:street}\par ${4:city}}
+	\backaddress{${5:$2}, ${6:$3}, ${7:$4}}
+	\place{${8:place}}
+
+	%\nowindowrules
+	%\nowindowtics
+	%\postremark{PLACE}
+	%\bottomtext{BOTTOMTEXT}
+
+	\signature{${9:$2}}
+
+	\subject{${10:$1}}
+
+	\date{`strftime("%A, %d %B %Y")`}
+
+	%\begin{dinquote}
+
+	\begin{letter}
+		{${11:recipient}\\
+		${12:street}\par
+		${13:city}}
+
+		%\phone{+49 4 21}{2 47 77 36}
+		%\yourmail{<+yourmail from ..+>}
+		%\sign{<+OURSIGN+>}
+
+		\opening{${14:opening}}
+
+		${16}
+
+		\closing{${15:closing}}
+		%\encl{<+ATTACHMENT+>}
+		%\ps{<+POSTSCRIPTUM+>}
+		%\cc{<+OTHER RECIPIENTS+>}
+	\end{letter}
+
+	%\end{dinquote}
+
+	\end{document}
