Index: vim/.vim/snippets/_.snippets
===================================================================
--- vim.orig/.vim/snippets/_.snippets	2009-12-20 13:13:32.525151451 +0100
+++ vim/.vim/snippets/_.snippets	2009-12-20 13:22:58.115183352 +0100
@@ -5,3 +5,90 @@
 	`&enc[:2] == "utf" ? "©" : "(c)"` Copyright `strftime("%Y")` ${1:`g:snips_author`}. All Rights Reserved.${2}
 snippet date
 	`strftime("%Y-%m-%d")`
+snippet http
+	http://${1:www.}${2}${3:.de}/${4}
+snippet dt
+	`strftime("%F %R")`
+snippet dl
+	`strftime("%d. %B %Y")`
+snippet rfcd
+	`strftime("%a, %d %b %Y %H:%M:%S %z")`
+snippet time
+	`strftime("%R")`
+snippet vi
+	`printf("vi: ft=%s:tw=%s:sw=%s:ts=%s", &filetype, &textwidth, &shiftwidth, &tabstop)`
+snippet www
+	`g:userWWW`
+snippet email
+	`g:userEmail`
+snippet phone
+	`g:userPhone`
+snippet addr
+	`g:userAddr`
+snippet city
+	`g:userCity`
+snippet zip
+	`g:userZip`
+snippet street
+	`g:userStreet`
+snippet fn
+	`g:userGivenName` `g:userName`
+snippet n
+	`g:userGivenName`
+snippet lg
+	Liebe Grüße,
+
+	${1}
+snippet mfg
+	Mit freundlichen Grüßen,
+
+	${1}
+snippet g
+	Grüße,
+
+	${1}
+snippet G
+	Greetings,
+
+	${1}
+snippet Kr
+	Kind Regards,
+
+	${1}
+snippet bug
+	Bug #
+snippet tick
+	Ticket #
+snippet ticket
+	Ticket #
+snippet gpl
+	# ${1:`Filename()`}
+	#  ${2:description}
+	#
+	# Depends: ${3:dependencies}
+	#
+	# Copyright (C) `strftime("%Y")` ${4:Copyright holder}
+	#
+	# http://www.${5:website}/
+	#
+	# All rights reserved.
+	#
+	# This program is free software; you can redistribute it and/or modify
+	# it under the terms of the GNU General Public License version 2 as
+	# published by the Free Software Foundation.
+	#
+	# Binary versions of this file provided by Univention to you as
+	# well as other copyrighted, protected or trademarked materials like
+	# Logos, graphics, fonts, specific documentations and configurations,
+	# cryptographic keys etc. are subject to a license agreement between
+	# you and Univention.
+	#
+	# This program is distributed in the hope that it will be useful,
+	# but WITHOUT ANY WARRANTY; without even the implied warranty of
+	# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	# GNU General Public License for more details.
+	#
+	# You should have received a copy of the GNU General Public License
+	# along with this program; if not, write to the Free Software
+	# Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+	${6}
