Index: vimconfig/.vim/snippets/_.snippets
===================================================================
--- vimconfig.orig/.vim/snippets/_.snippets	2009-10-22 09:01:46.000000000 +0200
+++ vimconfig/.vim/snippets/_.snippets	2009-10-22 09:04:55.000000000 +0200
@@ -5,3 +5,39 @@
 	`&enc[:2] == "utf" ? "©" : "(c)"` Copyright `strftime("%Y")` ${1:`g:snips_author`}. All Rights Reserved.${2}
 snippet date
 	`strftime("%Y-%m-%d")`
+snippet http
+	http://${1:www.}${2}${3:.de}/${4}
+snippet dt
+	`strftime("%F %R")`
+snippet dl
+	`strftime("%d. %B %Y")`
+snippet rfcd
+	`strftime("%a, %d %b %Y %H:%M:%S %z")`
+snippet time
+	`strftime("%R")`
+snippet vi
+	`printf("vi: ft=%s:tw=%s:sw=%s:ts=%s", &filetype, &textwidth, &shiftwidth, &tabstop)`
+snippet www
+	`g:userWWW`
+snippet email
+	`g:userEmail`
+snippet phone
+	`g:userPhone`
+snippet addr
+	`g:userAddr`
+snippet city
+	`g:userCity`
+snippet zip
+	`g:userZip`
+snippet street
+	`g:userStreet`
+snippet fn
+	`g:userGivenName` `g:userName`
+snippet n
+	`g:userGivenName`
+snippet bug
+	Bug #
+snippet tick
+	Ticket #
+snippet ticket
+	Ticket #
