Index: vimconfig/.vim/plugin/snipMate.vim
===================================================================
--- vimconfig.orig/.vim/plugin/snipMate.vim	2009-10-13 08:46:19.000000000 +0200
+++ vimconfig/.vim/plugin/snipMate.vim	2009-12-21 08:50:08.000000000 +0100
@@ -131,13 +131,13 @@
 		endif
 	endif
 
-	if pumvisible() " Update snippet if completion is used, or deal with supertab
-		if exists('SuperTabKey')
-			call feedkeys(SuperTabKey) | return ''
-		endif
-		call feedkeys("\<esc>a", 'n') " Close completion menu
-		call feedkeys("\<tab>") | return ''
-	endif
+	"if pumvisible() " Update snippet if completion is used, or deal with supertab
+	"	if exists('SuperTabKey')
+	"		call feedkeys(SuperTabKey) | return ''
+	"	endif
+	"	call feedkeys("\<esc>a", 'n') " Close completion menu
+	"	call feedkeys("\<tab>") | return ''
+	"endif
 
 	if exists('g:snipPos') | return snipMate#jumpTabStop(0) | endif
 
