Index: vim/.vim/addons/snipMate/plugin/snipMate.vim
===================================================================
--- vim.orig/.vim/addons/snipMate/plugin/snipMate.vim	2010-08-06 22:29:31.000000000 +0200
+++ vim/.vim/addons/snipMate/plugin/snipMate.vim	2010-08-07 00:27:44.000000000 +0200
@@ -53,13 +53,13 @@
 		endif
 	endif
 
-	if pumvisible() " Update snippet if completion is used, or deal with supertab
-		if exists('SuperTabKey')
-			call feedkeys(SuperTabKey) | return ''
-		endif
-		call feedkeys("\<esc>a", 'n') " Close completion menu
-		call feedkeys("\<tab>") | return ''
-	endif
+	"if pumvisible() " Update snippet if completion is used, or deal with supertab
+	"	if exists('SuperTabKey')
+	"		call feedkeys(SuperTabKey) | return ''
+	"	endif
+	"	call feedkeys("\<esc>a", 'n') " Close completion menu
+	"	call feedkeys("\<tab>") | return ''
+	"endif
 
 	if exists('g:snipPos') | return snipMate#jumpTabStop(0) | endif
 
