Index: .vim/bundle/supertab/plugin/supertab.vim
===================================================================
--- .vim.orig/bundle/supertab/plugin/supertab.vim	2014-03-25 08:54:04.550896443 +0100
+++ .vim/bundle/supertab/plugin/supertab.vim	2014-03-25 08:54:04.546896443 +0100
@@ -162,8 +162,8 @@
   " set the available completion types and modes.
   let s:types =
     \ "\<c-e>\<c-y>\<c-l>\<c-n>\<c-k>\<c-t>\<c-i>\<c-]>" .
-    \ "\<c-f>\<c-d>\<c-v>\<c-n>\<c-p>\<c-u>\<c-o>\<c-n>\<c-p>s"
-  let s:modes = '/^E/^Y/^L/^N/^K/^T/^I/^]/^F/^D/^V/^P/^U/^O/s'
+    \ "\<c-f>\<c-d>\<c-v>\<c-n>\<c-p>\<c-u>\<c-o>\<c-n>\<c-p>\<c-s>s"
+  let s:modes = '/^E/^Y/^L/^N/^K/^T/^I/^]/^F/^D/^V/^P/^U/^O/^S/s'
   let s:types = s:types . "np"
   let s:modes = s:modes . '/n/p'
 
