Index: .vim/ipi/tabular/plugin/Tabular.vim
===================================================================
--- .vim.orig/ipi/tabular/plugin/Tabular.vim	2013-01-10 20:57:56.740709705 +0100
+++ .vim/ipi/tabular/plugin/Tabular.vim	2013-01-10 21:01:12.176701956 +0100
@@ -55,9 +55,12 @@
 endfunction
 
 " Parse '/pattern/format' into separate pattern and format parts.         {{{2
+" If pattern is not present, use current search pattern
 " If parsing fails, return [ '', '' ]
 function! s:ParsePattern(string)
-  if a:string[0] != '/'
+  if empty(a:string)
+    return [@/,'']
+  elseif a:string[0] != '/'
     return ['','']
   endif
 
@@ -70,6 +73,14 @@
     let pattern = a:string[1 : -1]
   endif
 
+  if empty(pattern)
+    let pattern = @/
+  else
+    " assign pattern as search pattern to act like with other highly integrated
+    " commands, e.g. :g
+    let @/ = pattern
+  endif
+
   return [pattern, format]
 endfunction
 
