Index: .vim/bundle/taglist/plugin/taglist.vim
===================================================================
--- .vim.orig/bundle/taglist/plugin/taglist.vim	2010-12-19 23:27:27.114476909 +0100
+++ .vim/bundle/taglist/plugin/taglist.vim	2010-12-19 23:30:34.072950910 +0100
@@ -94,8 +94,8 @@
         elseif executable('tags')
             let Tlist_Ctags_Cmd = 'tags'
         else
-            echomsg 'Taglist: Exuberant ctags (http://ctags.sf.net) ' .
-                        \ 'not found in PATH. Plugin is not loaded.'
+			"echomsg 'Taglist: Exuberant ctags (http://ctags.sf.net) ' .
+						"\ 'not found in PATH. Plugin is not loaded.'
             " Skip loading the plugin
             let loaded_taglist = 'no'
             let &cpo = s:cpo_save
