Index: .vim/bundle/taglist/plugin/taglist.vim
===================================================================
--- .vim.orig/bundle/taglist/plugin/taglist.vim	2010-12-19 23:30:34.072950910 +0100
+++ .vim/bundle/taglist/plugin/taglist.vim	2010-12-19 23:30:34.928522911 +0100
@@ -981,7 +981,11 @@
 " Tlist_Get_Buffer_Filetype
 " Get the filetype for the specified buffer
 function! s:Tlist_Get_Buffer_Filetype(bnum)
-    let buf_ft = getbufvar(a:bnum, '&filetype')
+    let buf_fts = split(getbufvar(a:bnum, '&filetype'), '\.')
+    if len(buf_fts) == 0
+        let buf_fts = ['']
+    endif
+    let buf_ft = buf_fts[0]
 
     if bufloaded(a:bnum)
         " For loaded buffers, the 'filetype' is already determined
