Index: vimconfig/.vim/ftplugin/taskpaper.vim
===================================================================
--- vimconfig.orig/.vim/ftplugin/taskpaper.vim	2009-07-27 07:53:45.000000000 +0200
+++ vimconfig/.vim/ftplugin/taskpaper.vim	2009-07-27 07:54:00.000000000 +0200
@@ -7,10 +7,10 @@
 " Last Change:  2008-03-03
 
 
-if exists("loaded_task_paper")
+if exists("b:loaded_task_paper")
     finish
 endif
-let loaded_task_paper = 1
+let b:loaded_task_paper = 1
 
 "add '@' to keyword character set so that we can complete contexts as keywords
 setlocal iskeyword+=@-@
@@ -64,10 +64,12 @@
 endfunction
 
 " Set up mappings
+if mapcheck ('<Plug>ToggleDone') == ""
 noremap <unique> <script> <Plug>ToggleDone       :call <SID>ToggleDone()<CR>
 noremap <unique> <script> <Plug>ShowContext      :call <SID>ShowContext()<CR>
 noremap <unique> <script> <Plug>ShowAll          :call <SID>ShowAll()<CR>
 noremap <unique> <script> <Plug>FoldAllProjects  :call <SID>FoldAllProjects()<CR>
+endif
 
 map <buffer> <silent> <LocalLeader>td <Plug>ToggleDone
 map <buffer> <silent> <LocalLeader>tc <Plug>ShowContext
