Index: vim/.vim/plugin/tasklist.vim
===================================================================
--- vim.orig/.vim/plugin/tasklist.vim	2009-07-06 23:24:51.000000000 +0200
+++ vim/.vim/plugin/tasklist.vim	2009-07-06 23:25:00.000000000 +0200
@@ -365,9 +365,9 @@
 command! TaskList call s:TaskList()
 
 " Default key map
-if !hasmapto('<Plug>TaskList')
-    map <unique> <Leader>t <Plug>TaskList
-endif
+"if !hasmapto('<Plug>TaskList')
+"    map <unique> <Leader>t <Plug>TaskList
+"endif
 
 " Key map to Command
 nnoremap <unique> <script> <Plug>TaskList :TaskList<CR>
