Index: .vim/pack/submodules/start/terminus/plugin/terminus.vim
===================================================================
--- .vim.orig/pack/submodules/start/terminus/plugin/terminus.vim
+++ .vim/pack/submodules/start/terminus/plugin/terminus.vim
@@ -139,12 +139,19 @@ if s:paste
     " Disable bracketed paste mode on leaving Vim.
     let &t_te="\e[?2004l" . &t_te
 
-    set pastetoggle=<Esc>[201~
-    inoremap <expr> <Esc>[200~ terminus#private#paste('')
-    nnoremap <expr> <Esc>[200~ terminus#private#paste('i')
-    vnoremap <expr> <Esc>[200~ terminus#private#paste('c')
-    cnoremap <Esc>[200~ <nop>
-    cnoremap <Esc>[201~ <nop>
+    function! XTermPasteBegin(ret)
+        set pastetoggle=<f29>
+        set paste
+        return a:ret
+    endfunction
+
+    execute "set <f28>=\<Esc>[200~"
+    execute "set <f29>=\<Esc>[201~"
+    map <expr> <f28> XTermPasteBegin("i")
+    imap <expr> <f28> XTermPasteBegin("")
+    vmap <expr> <f28> XTermPasteBegin("c")
+    cmap <f28> <nop>
+    cmap <f29> <nop>
   endif
 endif
 
