disables imapping C-t
--- a/.vim/addons/Toggle/plugin/toggle.vim
+++ b/.vim/addons/Toggle/plugin/toggle.vim
@@ -52,9 +52,9 @@ let loaded_toggle=1
 let s:save_cpo = &cpo
 set cpo&vim
 
-imap <C-T> <C-O>:call Toggle()<CR>
-nmap <C-T> :call Toggle()<CR>
-vmap <C-T> <ESC>:call Toggle()<CR>
+"imap <C-T> <C-O>:call Toggle()<CR>
+nmap <ESC>t :call Toggle()<CR>
+vmap <ESC>t <ESC>:call Toggle()<CR>
 
 "--------------------------------------------------
 " If you don't want to break the standard <C-T> assignments,
