Index: .vim/bundle/Toggle/plugin/toggle.vim
===================================================================
--- .vim.orig/bundle/Toggle/plugin/toggle.vim	2010-12-19 23:27:26.490788910 +0100
+++ .vim/bundle/Toggle/plugin/toggle.vim	2010-12-19 23:30:38.886542910 +0100
@@ -67,7 +67,6 @@
 let s:save_cpo = &cpo
 set cpo&vim
 
-imap <C-T> <C-O>:call Toggle()<CR>
 nmap + :call Toggle()<CR>
 vmap + <ESC>:call Toggle()<CR>
 
