Index: vim/.vim/ftplugin/txt.vim
===================================================================
--- vim.orig/.vim/ftplugin/txt.vim	2010-06-07 15:26:43.109422688 +0200
+++ vim/.vim/ftplugin/txt.vim	2010-06-07 15:27:17.619390095 +0200
@@ -4,10 +4,19 @@
 " Last modified:	2010.06.06
 " License:		GPL
 
+if &cp || exists("g:loaded_txt")
+	finish
+endif
+let g:loaded_txt = 1
+
 " Line continuation used here
 let s:cpo_save = &cpo
 set cpo&vim
 
+if !exists('Tlist_Ctags_Cmd')
+	finish
+endif
+
 " ****************** Options *******************************************
 "How many title level to support, default is 3.
 if !exists('TxtBrowser_Title_Level')
