Index: vim/.vim/addons/TxtBrowser/ftplugin/txt.vim
===================================================================
--- vim.orig/.vim/addons/TxtBrowser/ftplugin/txt.vim	2010-08-05 15:46:58.000000000 +0200
+++ vim/.vim/addons/TxtBrowser/ftplugin/txt.vim	2010-08-26 21:50:16.000000000 +0200
@@ -4,10 +4,19 @@
 " Last modified:	2010.08.07
 " License:		GPL
 
+if &cp || exists("g:loaded_txt")
+	finish
+endif
+let g:loaded_txt = 1
+
 " Line continuation used here
 let s:cpo_save = &cpo
 set cpo&vim
 
+if !exists('Tlist_Ctags_Cmd')
+	finish
+endif
+
 " ****************** Options *******************************************
 "How many title level to support, default is 3.
 if !exists('TxtBrowser_Title_Level')
