Index: vimconfig/.vim/addons/TxtBrowser/ftplugin/txt.vim
===================================================================
--- vimconfig.orig/.vim/addons/TxtBrowser/ftplugin/txt.vim	2010-08-03 20:55:32.000000000 +0200
+++ vimconfig/.vim/addons/TxtBrowser/ftplugin/txt.vim	2010-08-03 20:55:48.000000000 +0200
@@ -4,10 +4,19 @@
 " Last modified:	2010.07.11
 " License:		GPL
 
+if &cp || exists("g:loaded_txt")
+	finish
+endif
+let g:loaded_txt = 1
+
 " Line continuation used here
 let s:cpo_save = &cpo
 set cpo&vim
 
+if !exists('Tlist_Ctags_Cmd')
+	finish
+endif
+
 " ****************** Options *******************************************
 "How many title level to support, default is 3.
 if !exists('TxtBrowser_Title_Level')
