Index: .vim/bundle/UltiSnips/UltiSnips/python.snippets
===================================================================
--- .vim.orig/bundle/UltiSnips/UltiSnips/python.snippets	2011-10-09 14:10:11.669071755 +0200
+++ .vim/bundle/UltiSnips/UltiSnips/python.snippets	2011-10-09 14:47:18.196859982 +0200
@@ -5,7 +5,7 @@
 #! header
 snippet #! "Shebang header for python scripts" b
 #!/usr/bin/env python
-# encoding: utf-8
+# `!v printf("-*- coding: %s -*-", &fileencoding != "" ? &fileencoding : "utf-8")`
 $0
 endsnippet
 
Index: .vim/bundle/UltiSnips/UltiSnips/sh.snippets
===================================================================
--- .vim.orig/bundle/UltiSnips/UltiSnips/sh.snippets	2011-10-09 14:10:11.693070980 +0200
+++ .vim/bundle/UltiSnips/UltiSnips/sh.snippets	2011-10-09 14:47:18.196859982 +0200
@@ -28,7 +28,7 @@
 endsnippet
 
 snippet elif "elif .. (elif)"
-elif ${2:[[ ${1:condition} ]]}; then
+elif ${2:[ ${1:condition} ]}; then
 	${0:#statements}
 endsnippet
 
