Index: vim/.vim/plugin/UltiSnips.vim
===================================================================
--- vim.orig/.vim/plugin/UltiSnips.vim	2009-07-26 12:47:40.000000000 +0200
+++ vim/.vim/plugin/UltiSnips.vim	2009-07-26 12:47:56.000000000 +0200
@@ -1,7 +1,7 @@
 " File: UltiSnips.vim
 " Author: Holger Rapp <SirVer@gmx.de>
 " Description: The Ultimate Snippets solution for Vim
-" Last Modified: July 21, 2009
+" Last Modified: Sun 26. Jul 2009 12:47:56 +0200 CEST
 "
 " Testing Info: {{{
 "   Running vim + ultisnips with the absolute bar minimum settings inside a   screen session:
@@ -10,7 +10,7 @@
 "     $ ./test.py  # launch the testsuite
 " }}}
 
-if exists('did_UltiSnips_vim') || &cp || version < 700
+if exists('did_UltiSnips_vim') || &cp || version < 700 || !has('python')
 	finish
 endif
 
