Index: nvim/lazy/vim-unimpaired/plugin/unimpaired.vim
===================================================================
--- nvim.orig/lazy/vim-unimpaired/plugin/unimpaired.vim
+++ nvim/lazy/vim-unimpaired/plugin/unimpaired.vim
@@ -622,8 +622,8 @@ endfunction
 
 exe UnimpairedMapTransform('string_encode','[y')
 exe UnimpairedMapTransform('string_decode',']y')
-exe UnimpairedMapTransform('string_encode','[C')
-exe UnimpairedMapTransform('string_decode',']C')
+" exe UnimpairedMapTransform('string_encode','[C')
+" exe UnimpairedMapTransform('string_decode',']C')
 exe UnimpairedMapTransform('url_encode','[u')
 exe UnimpairedMapTransform('url_decode',']u')
 exe UnimpairedMapTransform('xml_encode','[x')
