Index: nvim/packer/opt/vim-unimpaired/plugin/unimpaired.vim
===================================================================
--- nvim.orig/packer/opt/vim-unimpaired/plugin/unimpaired.vim
+++ nvim/packer/opt/vim-unimpaired/plugin/unimpaired.vim
@@ -598,4 +598,15 @@ exe UnimpairedMapTransform('url_decode',
 exe UnimpairedMapTransform('xml_encode','[x')
 exe UnimpairedMapTransform('xml_decode',']x')
 
+function! s:base64_encode(str) abort
+  return system("base64 -w0", a:str)
+endfunction
+
+function! s:base64_decode(str) abort
+  return system("base64 -d", a:str)
+endfunction
+
+call UnimpairedMapTransform('base64_encode','[Y')
+call UnimpairedMapTransform('base64_decode',']Y')
+
 " vim:set sw=2 sts=2:
