Index: .vim/bundle/VimCompletesMe/plugin/VimCompletesMe.vim
===================================================================
--- .vim.orig/bundle/VimCompletesMe/plugin/VimCompletesMe.vim
+++ .vim/bundle/VimCompletesMe/plugin/VimCompletesMe.vim
@@ -64,10 +64,10 @@ function! s:vim_completes_me(shift_tab)
   endif
 
   " First fallback to keyword completion if special completion was already tried.
-  if exists('b:completion_tried') && b:completion_tried
-    let b:completion_tried = 0
-    return "\<C-e>" . dirs[!dir]
-  endif
+  " if exists('b:completion_tried') && b:completion_tried
+  "   let b:completion_tried = 0
+  "   return "\<C-e>" . dirs[!dir]
+  " endif
 
   " Fallback
   let b:completion_tried = 1
