Index: .vim/bundle/vCoolor/plugin/vCoolor.vim
===================================================================
--- .vim.orig/bundle/vCoolor/plugin/vCoolor.vim
+++ .vim/bundle/vCoolor/plugin/vCoolor.vim
@@ -50,19 +50,19 @@ endfor
 " {
 let s:vcoolorMap = exists('g:vcoolor_map') ? g:vcoolor_map : '<A-c>'
 execute "nmap <silent> ".s:vcoolorMap." :VCoolor<CR>"
-execute "imap <silent> ".s:vcoolorMap." <Esc>:VCoolor<CR>a"
+" execute "imap <silent> ".s:vcoolorMap." <Esc>:VCoolor<CR>a"
 
 let s:vcoolInsRMap = exists('g:vcool_ins_rgb_map') ? g:vcool_ins_rgb_map : '<A-r>'
 execute "nmap <silent> ".s:vcoolInsRMap." :VCoolIns r<CR>"
-execute "imap <silent> ".s:vcoolInsRMap." <Esc>:VCoolIns r<CR>a"
+" execute "imap <silent> ".s:vcoolInsRMap." <Esc>:VCoolIns r<CR>a"
 
 let s:vcoolInsHMap = exists('g:vcool_ins_hsl_map') ? g:vcool_ins_hsl_map : '<A-v>'
 execute "nmap <silent> ".s:vcoolInsHMap." :silent VCoolIns h<CR>"
-execute "imap <silent> ".s:vcoolInsHMap." <Esc>:VCoolIns h<CR>a"
+" execute "imap <silent> ".s:vcoolInsHMap." <Esc>:VCoolIns h<CR>a"
 
 let s:vcoolInsRaMap = exists('g:vcool_ins_rgba_map') ? g:vcool_ins_rgba_map : '<A-w>'
 execute "nmap <silent> ".s:vcoolInsRaMap." :silent VCoolIns ra<CR>"
-execute "imap <silent> ".s:vcoolInsRaMap." <Esc>:VCoolIns ra<CR>a"
+" execute "imap <silent> ".s:vcoolInsRaMap." <Esc>:VCoolIns ra<CR>a"
 " }
 
 " VARIABLES
