Index: vimrc/pack/submodules/opt/vCoolor/plugin/vCoolor.vim
===================================================================
--- vimrc.orig/pack/submodules/opt/vCoolor/plugin/vCoolor.vim
+++ vimrc/pack/submodules/opt/vCoolor/plugin/vCoolor.vim
@@ -164,7 +164,10 @@ function s:SetColorByType(oldColor, newC
     let l:oldCol = a:oldColor[0]
     let l:newCol = strpart(a:newCol, 0, 7) " To remove the ******* annoying ^@.
 
-    if a:oldColor[2] == 'r'
+    if a:oldColor[2] =~ '^rgb('
+        let s:newCol = vcoolor#convert#Rgb2Hex(substitute(substitute(s:newCol, '^rgb(', '', ''), ')', '', ''))
+        execute "silent: s/".l:oldCol."/".l:newCol
+    elseif a:oldColor[2] == 'r'
         let l:newCol = vcoolor#convert#Hex2Rgb(l:newCol)
         execute "silent: s/".l:oldCol."/".l:newCol
 	elseif a:oldColor[2] == 'rgba'
