Index: .vim/pack/submodules/start/vCoolor/plugin/vCoolor.vim
===================================================================
--- .vim.orig/pack/submodules/start/vCoolor/plugin/vCoolor.vim
+++ .vim/pack/submodules/start/vCoolor/plugin/vCoolor.vim
@@ -57,22 +57,22 @@ let s:vcoolInsRaMap = exists('g:vcool_in
 
 if !exists('g:vcoolor_disable_mappings') || exists('g:vcoolor_map')
 	execute "nmap <silent> ".s:vcoolorMap." :VCoolor<CR>"
-	execute "imap <silent> ".s:vcoolorMap." <Esc>:VCoolor<CR>a"
+	" execute "imap <silent> ".s:vcoolorMap." <Esc>:VCoolor<CR>a"
 endif
 
 if !exists('g:vcoolor_disable_mappings') || exists('g:vcool_ins_rgb_map')
 	execute "nmap <silent> ".s:vcoolInsRMap." :VCoolIns r<CR>"
-	execute "imap <silent> ".s:vcoolInsRMap." <Esc>:VCoolIns r<CR>a"
+	" execute "imap <silent> ".s:vcoolInsRMap." <Esc>:VCoolIns r<CR>a"
 endif
 
 if !exists('g:vcoolor_disable_mappings') || exists('g:vcool_ins_hsl_map')
 	execute "nmap <silent> ".s:vcoolInsHMap." :silent VCoolIns h<CR>"
-	execute "imap <silent> ".s:vcoolInsHMap." <Esc>:VCoolIns h<CR>a"
+	" execute "imap <silent> ".s:vcoolInsHMap." <Esc>:VCoolIns h<CR>a"
 endif
 
 if !exists('g:vcoolor_disable_mappings') || exists('g:vcool_ins_rgba_map')
 	execute "nmap <silent> ".s:vcoolInsRaMap." :silent VCoolIns ra<CR>"
-	execute "imap <silent> ".s:vcoolInsRaMap." <Esc>:VCoolIns ra<CR>a"
+	" execute "imap <silent> ".s:vcoolInsRaMap." <Esc>:VCoolIns ra<CR>a"
 endif
 " }}}
 
