Index: .vim/bundle/yankstack/autoload/yankstack.vim
===================================================================
--- .vim.orig/bundle/yankstack/autoload/yankstack.vim
+++ .vim/bundle/yankstack/autoload/yankstack.vim
@@ -11,7 +11,11 @@ let s:last_paste = { 'changedtick': -1,
 
 function! s:yank_with_key(key)
   call s:before_yank()
-  return a:key
+  if a:key ==# 'Y'
+	  return 'y$'
+  else
+	  return a:key
+  endif
 endfunction
 
 function! s:paste_with_key(key, mode, register, count)
